{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "4547774c-87c6-4ee4-811b-d4ba8a66a3a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "###calculate the dN/dS ratio of the genomes\n",
    "###convert protein alignment into codon aligenment using pal2nal\n",
    "###need perl package\n",
    "from Bio import SeqIO\n",
    "import re\n",
    "import glob\n",
    "import os "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "56605480-2056-433c-ab7a-0e04dbc21248",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "整合完成，结果保存在 '/mnt/maple/ymchen/gene_loss/preflow/cds_dataset.fna'\n"
     ]
    }
   ],
   "source": [
    "#build CDS dataset of the genomes set\n",
    "#,target_file)\n",
    "def get_full_gbk(proka_result,target_file):\n",
    "    pattern = os.path.join(proka_result, '**', '*.ffn')\n",
    "    fnn_files = glob.glob(pattern, recursive=True)\n",
    "    with open(target_file, 'w') as outfile:\n",
    "        for file in fnn_files:\n",
    "            with open(file, 'r') as infile:\n",
    "                records = SeqIO.parse(infile, 'fasta')\n",
    "                SeqIO.write(records, outfile, 'fasta')\n",
    "    print(f\"整合完成，结果保存在 '{target_file}'\")\n",
    "get_full_gbk(\"/mnt/maple/ymchen/gene_loss/preflow/prokka_result\",\"/mnt/maple/ymchen/gene_loss/preflow/cds_dataset.fna\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "45922493-41d4-4165-8e2e-1d975c79397e",
   "metadata": {},
   "outputs": [],
   "source": [
    "#sequence_file,target_file\n",
    "def get_corresponding_cds(protein_file,dataset,target_file):\n",
    "    id_list=[]\n",
    "    container={}\n",
    "    with open(protein_file) as p:\n",
    "        for line in p:\n",
    "            line=line.strip()\n",
    "            if bool(re.search(r'>', line)):\n",
    "                id = line[1:]\n",
    "                id_list.append(id)\n",
    "        #print(id_list)\n",
    "    with open(dataset, 'r') as infile, open(target_file, 'w') as outfile:\n",
    "        for record in SeqIO.parse(infile, 'fasta'):\n",
    "            if record.id in id_list:\n",
    "                container[record.id]=record\n",
    "        ordered_container = {key: container[key] for key in id_list if key in container}\n",
    "        for seq_id,sequence in ordered_container.items():\n",
    "            SeqIO.write(sequence, outfile, 'fasta')\n",
    "    #print(\"提取完成，结果保存在 'extracted_nucleotides.fasta'\")\n",
    "#get_corresponding_cds(\"/mnt/maple/ymchen/gene_loss/preflow/ortho/Results_Sep11/Single_Copy_Orthologue_Sequences/OG0001442.fa\",\"/mnt/maple/ymchen/gene_loss/preflow/cds_dataset.fna\",\"/mnt/maple/ymchen/gene_loss/preflow/example.fna\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "cf5e0899-44ec-419d-8618-d04725332796",
   "metadata": {},
   "outputs": [],
   "source": [
    "### get full folder of correponding files \n",
    "def get_folder_correspond(ortholog_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(ortholog_folder):\n",
    "        true_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(ortholog_folder, filename)\n",
    "        get_corresponding_cds(file_path,\"/mnt/maple/ymchen/gene_loss/preflow/cds_dataset.fna\",f\"{target_folder}/{true_name}.fna\")\n",
    "        \n",
    "get_folder_correspond(\"/mnt/maple/ymchen/gene_loss/preflow/ortho/Results_Sep11/Single_Copy_Orthologue_Sequences/\",\"/mnt/maple/ymchen/gene_loss/preflow/correspond_files\")  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "e8e9885f-ebb3-4db6-8901-16978c6e6732",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001303.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001303.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001234.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001234.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001303.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001303.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001303.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001234.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001234.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001234.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001351.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001351.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001296.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001296.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001351.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001351.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001351.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001296.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001296.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001296.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001338.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001338.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000910.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000910.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001338.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001338.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001338.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000910.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000910.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000910.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001104.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001104.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001104.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001104.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001104.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001021.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001021.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001021.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001021.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001021.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000936.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000936.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001154.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001154.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000936.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000936.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000936.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001154.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001154.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001154.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001141.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001141.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001246.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001246.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001141.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001141.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001141.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001246.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001246.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001246.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001230.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001230.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001438.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001438.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001230.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001230.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001230.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001438.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001438.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001438.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001185.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001185.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000828.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000828.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001185.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001185.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001185.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000828.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000828.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000828.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000818.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000818.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001146.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001146.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000818.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000818.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000818.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001146.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001146.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001146.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000983.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000983.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000985.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000985.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001394.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001394.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000983.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000983.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000983.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000985.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000985.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000985.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001394.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001394.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001394.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001003.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001003.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001003.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001003.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001003.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001041.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001041.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001370.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001370.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001041.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001041.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001041.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001370.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001370.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001370.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001332.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001332.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000928.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000928.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001332.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001332.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001332.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000928.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000928.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000928.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000808.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000808.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000808.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000808.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000808.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001148.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001148.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001148.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001148.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001148.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001200.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001200.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001200.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001200.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001200.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001229.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001229.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001229.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001229.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001229.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001189.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001189.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001189.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001189.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001189.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000911.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000911.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000911.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001356.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001356.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001356.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000911.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000911.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001356.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001356.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001136.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001136.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001198.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001198.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001136.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001136.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001136.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001198.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001198.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001198.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001040.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001040.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001006.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001006.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001040.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001040.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001040.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001006.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001006.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001006.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000900.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000900.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000900.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000900.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000900.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001392.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001392.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001392.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001392.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001392.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001379.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001379.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001379.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001379.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001379.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001197.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001197.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001197.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001197.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001197.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001381.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001381.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001381.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001381.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001381.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000991.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000991.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000991.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000991.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000991.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000978.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000978.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000978.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000978.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000978.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000814.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000814.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000814.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000814.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000814.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001103.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001103.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001103.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001103.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001103.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000827.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000827.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000827.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000827.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000827.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001398.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001398.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001398.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001108.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001108.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001108.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001398.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001398.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001108.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001108.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001386.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001386.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001386.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001386.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001386.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000811.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000811.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000811.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000811.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000811.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001408.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001408.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001408.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001408.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001408.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001045.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001045.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000841.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000841.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001045.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001045.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001045.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000841.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000841.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000841.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000864.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000864.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000864.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000864.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000864.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001314.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001314.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001149.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001149.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001314.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001314.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001314.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001149.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001149.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001149.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000815.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000815.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001359.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001359.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000815.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000815.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000815.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001359.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001359.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001359.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000970.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000970.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000970.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000970.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000970.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001431.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001431.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001431.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001431.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001431.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001000.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001000.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001000.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001000.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001000.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001271.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001271.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000918.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000918.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001053.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001053.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001271.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001271.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001271.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000918.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000918.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000918.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001053.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001053.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001053.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000971.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000971.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000971.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000971.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000971.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000813.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000813.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001241.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001241.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000813.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000813.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000813.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001241.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001241.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001241.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001249.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001249.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001249.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001249.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001249.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001191.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001191.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001191.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001191.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001191.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000850.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000850.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001235.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001235.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000850.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000850.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000850.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001235.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001235.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001235.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001070.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001070.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000810.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000810.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001035.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001035.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001070.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001070.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001070.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000810.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000810.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000810.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001035.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001035.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001035.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001244.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001244.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000887.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000887.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001244.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001244.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001244.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000887.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000887.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000887.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000809.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000809.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001151.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001151.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000809.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000809.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000809.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001151.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001151.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001151.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001220.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001220.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001220.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001220.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001220.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000923.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000923.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000824.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000824.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000923.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000923.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000923.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000824.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000824.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000824.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001396.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001396.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001396.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001396.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001396.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001025.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001025.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001353.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001353.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001025.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001025.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001025.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001353.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001353.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001353.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000935.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000935.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000938.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000938.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000935.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000935.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000935.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000938.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000938.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000938.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001433.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001433.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001433.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001433.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001433.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001135.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001135.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001135.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001135.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001135.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000976.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000976.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001268.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001268.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000976.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000976.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000976.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001268.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001268.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001268.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001100.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001100.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001100.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001100.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001100.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001029.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001029.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001304.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001304.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001029.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001029.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001029.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001304.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001304.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001304.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000876.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000876.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000905.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000905.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000876.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000876.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000876.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000905.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000905.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000905.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001106.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001106.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001329.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001329.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001106.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001106.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001106.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001329.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001329.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001329.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001127.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001127.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000825.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000825.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001127.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001127.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001127.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000825.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000825.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000825.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001279.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001279.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001395.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001395.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001279.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001279.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001279.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001395.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001395.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001395.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001432.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001432.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000847.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000847.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001432.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001432.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001432.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000847.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000847.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000847.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001131.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001131.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001347.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001347.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000807.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000807.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001131.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001131.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001131.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001347.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001347.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001347.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000807.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000807.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000807.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000989.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000989.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001017.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001017.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000989.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000989.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000989.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001017.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001017.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001017.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001321.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001321.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001164.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001164.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001321.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001321.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001321.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001164.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001164.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001164.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001037.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001037.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001037.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001037.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001037.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001165.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001165.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001165.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001165.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001165.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000981.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000981.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001095.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001095.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000981.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000981.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000981.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001095.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001095.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001095.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000822.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000822.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001071.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001071.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001292.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001292.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000822.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000822.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000822.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001071.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001071.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001071.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001292.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001292.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001292.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001160.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001160.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001087.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001087.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001160.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001160.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001160.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001087.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001087.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001087.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001067.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001067.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001344.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001344.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001067.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001067.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001067.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001344.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001344.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001344.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000921.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000921.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001274.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001274.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000921.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000921.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000921.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001274.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001274.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001274.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001099.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001099.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001266.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001266.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001436.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001436.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001099.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001099.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001099.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001266.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001266.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001266.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001436.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001436.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001436.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001009.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001009.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001391.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001391.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001364.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001364.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001009.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001009.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001009.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001391.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001391.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001391.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001364.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001364.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001364.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001243.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001243.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001096.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001096.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001243.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001243.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001243.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001096.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001096.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001096.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001107.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001107.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001010.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001010.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001107.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001107.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001107.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001010.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001010.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001010.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001404.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001404.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001168.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001168.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001404.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001404.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001404.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001168.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001168.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001168.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000803.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000803.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000803.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000803.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000803.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000927.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000927.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000805.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000805.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000927.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000927.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000927.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000805.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000805.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000805.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000993.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000993.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000859.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000859.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000993.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000993.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000993.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000859.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000859.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000859.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000969.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000969.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000912.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000912.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000969.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000969.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000969.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000912.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000912.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000912.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001222.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001222.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000826.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000826.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001222.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001222.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001222.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000826.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000826.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000826.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001173.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001173.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000972.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000972.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001173.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001173.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001173.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000972.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000972.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000972.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001297.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001297.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001297.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001297.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001297.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000880.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000880.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000865.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000865.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001090.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001090.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000880.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000880.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000880.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000865.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000865.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000865.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001090.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001090.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001090.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000990.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000990.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000990.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000990.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000990.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001074.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001074.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000804.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000804.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001074.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001074.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001074.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000804.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000804.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000804.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001066.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001066.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000925.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000925.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001066.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001066.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001066.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000925.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000925.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000925.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000836.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000836.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000857.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000857.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000836.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000836.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000836.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000857.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000857.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000857.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001341.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001341.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000930.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000930.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001341.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001341.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001341.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000930.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000930.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000930.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001311.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001311.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001145.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001145.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001311.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001311.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001311.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001145.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001145.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001145.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001282.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001282.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001119.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001119.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001282.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001282.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001282.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001119.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001119.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001119.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001312.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001312.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001312.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001312.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001312.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001276.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001276.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001276.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001276.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001276.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001144.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001144.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001144.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001144.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001144.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001285.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001285.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001285.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001285.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001285.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000848.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000848.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000848.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000848.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000848.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001199.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001199.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001199.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001199.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001199.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001419.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001419.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001331.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001331.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001410.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001410.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001419.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001419.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001419.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001331.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001331.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001331.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001410.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001410.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001410.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001342.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001342.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000953.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000953.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001342.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001342.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001342.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000953.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000953.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000953.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001259.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001259.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001272.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001272.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001259.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001259.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001259.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001272.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001272.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001272.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001291.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001291.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001002.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001002.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001291.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001291.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001291.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001002.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001002.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001002.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001020.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001020.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001369.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001369.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001020.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001020.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001020.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001369.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001369.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001369.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001286.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001286.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001286.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001286.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001286.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001248.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001248.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001248.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001248.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001248.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000942.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000942.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000942.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000855.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000855.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000855.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000942.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000942.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000855.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000855.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001193.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001193.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000946.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000946.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001193.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001193.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001193.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000946.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000946.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000946.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001415.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001415.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001415.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001415.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001415.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001352.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001352.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001352.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001352.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001352.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001375.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001375.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001375.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001375.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001375.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001358.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001358.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001358.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001358.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001358.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000832.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000832.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000858.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000858.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000832.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000832.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000832.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000858.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000858.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000858.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000862.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000862.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001354.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001354.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000862.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000862.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000862.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001354.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001354.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001354.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001091.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001091.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001429.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001429.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001091.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001091.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001091.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001429.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001429.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001429.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000939.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000939.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001139.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001139.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000939.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000939.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000939.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001139.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001139.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001139.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001016.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001016.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001016.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001016.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001016.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001132.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001132.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001132.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001132.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001132.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001212.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001212.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001212.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001212.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001212.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001218.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001218.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001218.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001218.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001218.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001263.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001263.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001263.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001263.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001263.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001081.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001081.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001081.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001081.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001081.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001031.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001031.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001031.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001031.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001031.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000909.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000909.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000909.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000909.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000909.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001287.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001287.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001287.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001287.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001287.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001337.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001337.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001337.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001337.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001337.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000898.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000898.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000898.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000898.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000898.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000897.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000897.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001348.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001348.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000897.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000897.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000897.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001348.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001348.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001348.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000999.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000999.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001413.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001413.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000999.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000999.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000999.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001413.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001413.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001413.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000831.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000831.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001112.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001112.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000831.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000831.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000831.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001112.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001112.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001112.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001280.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001280.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000823.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000823.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001280.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001280.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001280.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000823.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000823.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000823.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001273.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001273.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001273.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001273.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001273.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000866.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000866.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000866.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000866.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000866.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000919.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000919.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000919.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000919.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000919.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001324.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001324.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001324.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001324.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001324.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001170.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001170.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001170.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001170.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001170.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000982.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000982.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001102.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001102.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000982.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000982.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000982.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001102.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001102.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001102.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000845.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000845.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000845.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000845.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000845.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001365.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001365.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001365.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001365.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001365.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001284.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001284.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001138.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001138.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001308.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001308.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001284.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001284.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001284.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001138.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001138.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001138.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001308.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001308.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001308.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000896.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000896.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001196.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001196.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000896.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000896.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000896.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001196.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001196.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001196.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001004.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001004.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001054.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001054.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001004.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001004.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001004.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001054.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001054.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001054.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001121.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001121.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001376.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001376.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001121.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001121.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001121.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001376.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001376.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001376.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001142.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001142.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000940.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000940.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001142.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001142.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001142.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000940.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000940.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000940.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000885.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000885.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000885.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000885.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000885.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001440.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001440.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001440.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001440.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001440.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000979.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000979.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000979.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000979.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000979.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001072.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001072.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001072.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001072.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001072.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000996.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000996.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000996.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000996.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000996.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000947.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000947.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000947.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000947.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000947.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001134.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001134.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001134.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001134.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001134.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001217.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001217.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001217.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001217.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001217.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000977.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000977.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000977.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001377.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001377.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001377.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000977.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000977.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001377.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001377.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000890.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000890.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000890.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000890.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000890.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000873.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000873.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000873.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000873.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000873.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001178.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001178.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001178.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001178.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001178.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001366.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001366.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001366.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001366.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001366.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001427.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001427.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001427.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001427.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001427.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001320.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001320.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001320.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001320.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001320.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001423.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001423.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001423.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001423.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001423.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001421.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001421.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001421.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001421.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001421.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000929.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000929.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001140.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001140.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000929.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000929.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000929.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001140.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001140.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001140.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000886.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000886.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000886.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000886.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000886.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001385.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001385.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001385.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001385.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001385.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001269.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001269.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001315.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001315.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000871.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000871.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001269.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001269.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001269.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001315.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001315.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001315.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000871.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000871.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000871.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000963.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000963.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000920.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000920.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001042.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001042.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000963.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000963.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000963.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000920.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000920.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000920.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001042.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001042.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001042.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001228.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001228.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000816.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000816.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001228.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001228.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001228.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000816.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000816.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000816.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001209.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001209.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001209.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001209.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001209.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001412.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001412.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001260.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001260.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001412.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001412.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001412.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001260.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001260.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001260.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000833.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000833.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000812.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000812.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000833.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000833.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000833.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000812.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000812.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000812.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001115.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001115.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001115.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001115.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001115.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001378.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001378.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001378.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001378.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001378.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001101.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001101.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001101.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001101.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001101.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001027.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001027.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001027.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001027.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001027.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001428.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001428.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001428.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001428.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001428.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000961.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000961.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000961.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000961.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000961.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001317.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001317.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001317.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001317.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001317.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001011.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001011.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001110.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001110.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000955.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000955.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001011.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001011.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001011.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001110.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001110.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001110.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000955.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000955.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000955.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001182.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001182.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001252.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001252.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001182.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001182.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001182.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001252.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001252.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001252.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001089.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001089.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000899.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000899.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001159.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001159.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001089.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001089.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001089.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000899.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000899.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000899.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001159.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001159.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001159.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001387.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001387.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001283.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001283.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001387.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001387.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001387.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001283.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001283.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001283.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000895.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000895.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000941.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000941.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000895.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000895.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000895.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000941.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000941.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000941.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001063.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001063.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001114.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001114.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001383.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001383.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001063.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001063.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001063.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001114.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001114.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001114.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001383.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001383.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001383.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000893.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000893.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000950.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000950.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000893.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000893.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000893.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000950.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000950.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000950.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001295.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001295.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001299.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001299.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001295.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001295.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001295.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001299.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001299.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001299.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001328.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001328.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001328.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001328.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001328.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001400.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001400.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001414.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001414.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001400.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001400.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001400.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001414.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001414.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001414.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001156.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001156.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000838.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000838.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001156.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001156.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001156.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000838.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000838.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000838.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001215.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001215.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001215.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001215.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001215.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001059.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001059.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001309.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001309.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001059.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001059.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001059.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001309.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001309.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001309.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001007.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001007.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001128.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001128.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001022.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001022.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001007.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001007.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001007.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001128.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001128.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001128.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001022.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001022.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001022.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001319.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001319.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001319.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001319.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001319.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000878.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000878.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001208.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001208.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000878.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000878.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000878.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001208.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001208.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001208.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000902.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000902.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001223.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001223.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000902.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000902.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000902.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001223.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001223.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001223.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001301.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001301.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001254.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001254.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001301.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001301.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001301.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001254.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001254.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001254.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001257.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001257.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001374.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001374.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001257.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001257.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001257.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001374.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001374.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001374.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001109.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001109.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001203.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001203.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000856.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000856.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001109.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001109.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001109.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001203.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001203.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001203.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000856.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000856.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000856.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001049.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001049.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001049.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001049.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001049.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000870.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000870.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000870.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000870.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000870.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001077.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001077.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001077.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001077.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001077.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000984.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000984.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000984.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000984.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000984.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001086.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001086.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001086.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001313.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001313.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001313.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001086.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001086.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001313.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001313.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000915.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000915.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000915.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000915.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000915.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000998.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000998.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000998.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000998.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000998.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000943.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000943.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000943.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000943.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000943.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001240.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001240.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001240.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001240.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001240.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001367.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001367.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001367.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001367.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001367.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001171.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001171.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001401.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001401.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001171.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001171.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001171.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001401.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001401.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001401.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001245.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001245.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000917.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000917.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001245.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001245.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001245.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000917.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000917.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000917.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001092.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001092.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001224.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001224.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001092.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001092.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001092.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001224.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001224.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001224.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001227.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001227.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001380.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001380.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001227.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001227.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001227.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001380.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001380.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001380.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001024.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001024.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001001.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001001.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001024.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001024.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001024.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001001.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001001.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001001.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001175.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001175.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000883.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000883.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001175.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001175.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001175.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000883.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000883.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000883.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001277.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001277.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001270.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001270.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001277.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001277.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001277.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001270.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001270.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001270.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001226.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001226.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001018.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001018.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000975.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000975.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001226.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001226.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001226.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001018.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001018.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001018.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000975.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000975.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000975.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001265.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001265.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000906.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000906.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001265.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001265.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001265.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000906.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000906.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000906.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000931.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000931.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000932.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000932.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000931.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000931.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000931.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000932.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000932.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000932.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001190.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001190.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001399.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001399.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001190.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001190.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001190.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001399.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001399.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001399.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000834.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000834.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001051.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001051.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000834.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000834.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000834.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001051.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001051.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001051.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001343.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001343.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001057.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001057.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001343.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001343.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001343.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001057.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001057.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001057.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001417.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001417.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001261.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001261.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001417.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001417.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001417.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001261.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001261.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001261.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000835.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000835.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000965.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000965.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001294.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001294.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000835.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000835.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000835.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000965.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000965.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000965.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001294.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001294.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001294.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000957.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000957.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001306.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001306.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000957.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000957.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000957.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001306.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001306.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001306.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001120.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001120.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001125.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001125.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001065.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001065.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001120.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001120.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001120.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001125.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001125.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001125.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001065.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001065.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001065.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001064.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001064.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001363.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001363.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001064.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001064.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001064.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001363.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001363.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001363.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001253.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001253.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001094.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001094.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001253.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001253.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001253.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001094.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001094.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001094.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001105.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001105.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000967.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000967.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001105.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001105.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001105.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000967.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000967.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000967.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001050.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001050.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000914.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000914.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001050.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001050.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001050.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000914.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000914.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000914.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000881.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000881.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000881.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000881.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000881.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000951.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000951.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001316.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001316.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000951.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000951.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000951.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001316.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001316.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001316.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001439.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001439.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001439.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001439.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001439.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000872.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000872.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001289.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001289.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000872.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000872.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000872.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001289.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001289.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001289.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000974.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000974.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000974.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000974.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000974.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000937.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000937.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001085.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001085.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001389.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001389.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000937.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000937.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000937.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001085.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001085.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001085.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001389.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001389.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001389.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001288.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001288.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001278.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001278.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001288.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001288.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001288.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001278.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001278.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001278.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000888.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000888.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001043.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001043.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000888.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000888.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000888.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001043.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001043.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001043.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001069.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001069.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000916.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000916.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001069.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001069.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001069.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000916.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000916.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000916.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001172.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001172.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001424.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001424.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001172.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001172.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001172.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001424.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001424.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001424.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001129.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001129.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001137.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001137.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001129.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001129.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001129.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001137.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001137.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001137.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001345.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001345.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001382.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001382.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001345.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001345.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001345.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001382.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001382.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001382.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001111.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001111.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001188.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001188.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001111.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001111.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001111.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001188.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001188.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001188.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001204.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001204.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000851.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000851.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001204.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001204.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001204.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000851.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000851.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000851.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001184.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001184.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000973.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000973.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001184.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001184.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001184.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000973.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000973.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000973.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001056.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001056.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001056.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001056.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001056.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001033.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001033.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001033.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001033.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001033.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001216.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001216.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001216.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001216.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001216.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001355.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001355.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000980.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000980.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001355.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001355.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001355.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000980.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000980.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000980.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001116.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001116.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001310.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001310.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001116.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001116.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001116.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001310.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001310.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001310.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001442.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001442.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001442.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001442.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001442.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001336.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001336.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000806.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000806.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001258.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001258.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001336.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001336.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001336.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000806.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000806.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000806.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001258.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001258.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001258.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000944.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000944.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000849.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000849.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000944.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000944.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000944.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000849.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000849.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000849.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001078.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001078.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001403.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001403.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001078.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001078.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001078.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001403.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001403.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001403.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001055.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001055.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001157.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001157.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001055.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001055.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001055.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001157.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001157.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001157.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000949.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000949.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001434.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001434.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001158.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001158.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000949.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000949.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000949.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001434.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001434.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001434.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001158.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001158.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001158.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000913.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000913.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000867.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000867.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000913.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000913.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000913.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000867.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000867.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000867.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000891.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000891.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000891.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000891.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000891.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000868.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000868.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000868.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000868.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000868.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001058.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001058.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001058.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001058.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001058.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001205.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001205.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001205.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001205.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001205.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001161.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001161.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001161.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001422.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001422.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001422.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001161.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001161.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001422.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001422.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001023.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001023.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001097.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001097.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001023.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001023.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001023.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001097.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001097.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001097.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000869.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000869.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001174.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001174.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000869.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000869.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000869.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001174.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001174.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001174.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001214.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001214.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001118.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001118.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000853.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000853.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001214.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001214.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001214.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001118.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001118.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001118.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000853.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000853.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000853.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001211.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001211.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000837.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000837.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001211.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001211.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001211.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000837.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000837.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000837.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001430.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001430.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001210.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001210.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001430.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001430.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001430.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001210.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001210.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001210.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001192.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001192.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001242.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001242.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001192.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001192.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001192.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001242.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001242.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001242.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001390.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001390.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000966.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000966.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001390.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001390.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001390.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000966.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000966.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000966.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001169.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001169.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001232.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001232.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001169.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001169.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001169.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001232.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001232.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001232.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001409.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001409.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001221.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001221.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001327.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001327.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001409.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001409.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001409.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001221.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001221.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001221.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001327.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001327.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001327.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001126.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001126.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001061.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001061.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001126.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001126.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001126.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001061.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001061.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001061.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000882.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000882.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001238.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001238.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000882.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000882.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000882.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001238.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001238.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001238.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001237.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001237.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000968.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000968.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001237.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001237.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001237.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000968.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000968.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000968.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001225.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001225.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001225.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001225.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001225.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001346.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001346.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001117.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001117.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001346.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001346.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001346.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001117.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001117.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001117.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000995.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000995.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000846.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000846.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000875.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000875.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000995.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000995.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000995.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000846.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000846.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000846.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000875.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000875.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000875.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001323.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001323.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000997.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000997.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001323.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001323.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001323.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000997.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000997.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000997.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001368.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001368.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001339.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001339.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001368.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001368.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001368.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001339.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001339.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001339.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000987.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000987.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001080.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001080.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000987.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000987.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000987.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001080.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001080.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001080.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001019.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001019.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001318.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001318.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000829.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000829.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001019.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001019.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001019.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001318.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001318.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001318.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000829.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000829.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000829.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001305.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001305.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001181.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001181.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001034.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001034.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001305.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001305.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001305.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001181.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001181.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001181.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001034.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001034.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001034.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000962.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000962.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001152.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001152.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000962.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000962.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000962.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001152.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001152.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001152.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000964.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000964.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000964.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000964.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000964.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001062.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001062.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001062.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001062.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001062.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000988.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000988.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000988.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000988.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000988.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001334.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001334.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001307.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001307.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001334.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001334.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001334.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001307.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001307.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001307.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000954.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000954.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001113.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001113.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001186.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001186.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000954.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000954.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000954.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001113.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001113.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001113.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001186.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001186.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001186.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001183.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001183.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000892.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000892.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001183.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001183.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001183.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000892.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000892.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000892.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001425.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001425.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001425.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001425.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001425.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001028.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001028.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001255.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001255.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001028.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001028.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001028.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001255.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001255.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001255.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001076.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001076.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001076.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001076.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001076.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000994.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000994.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000994.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000994.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000994.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001068.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001068.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001068.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001068.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001068.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000922.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000922.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000922.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000922.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000922.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000839.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000839.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000839.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000839.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000839.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001326.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001326.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001326.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001326.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001326.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001207.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001207.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001124.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001124.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001207.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001207.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001207.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001124.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001124.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001124.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000863.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000863.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000863.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000863.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000863.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001008.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001008.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001008.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001008.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001008.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001162.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001162.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001162.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001162.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001162.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001250.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001250.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001250.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001250.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001250.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000874.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000874.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000874.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000874.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000874.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001079.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001079.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001079.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001079.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001079.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001155.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001155.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001239.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001239.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001155.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001155.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001155.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001239.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001239.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001239.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000842.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000842.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001290.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001290.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001179.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001179.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000842.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000842.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000842.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001290.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001290.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001290.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001179.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001179.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001179.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001262.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001262.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001330.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001330.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001262.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001262.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001262.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001330.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001330.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001330.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000908.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000908.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001088.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001088.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000908.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000908.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000908.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001088.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001088.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001088.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001397.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001397.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001397.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001397.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001397.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001373.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001373.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001373.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001373.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001373.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001302.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001302.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001302.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001302.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001302.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001420.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001420.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001420.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001420.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001420.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001437.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001437.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001437.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001437.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001437.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000877.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000877.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001153.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001153.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000877.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000877.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000877.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001153.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001153.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001153.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001402.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001402.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001322.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001322.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001402.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001402.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001402.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001322.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001322.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001322.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001388.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001388.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001333.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001333.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001388.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001388.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001388.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001333.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001333.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001333.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000879.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000879.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000879.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000879.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000879.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001360.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001360.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001360.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001360.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001360.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000860.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000860.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001411.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001411.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000860.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000860.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000860.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001411.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001411.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001411.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001426.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001426.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001122.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001122.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001426.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001426.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001426.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001122.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001122.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001122.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000840.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000840.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001026.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001026.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000840.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000840.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000840.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001026.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001026.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001026.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001384.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001384.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001384.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001384.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001384.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001416.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001416.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001416.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001416.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001416.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001340.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001340.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001340.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001340.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001340.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000901.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000901.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001133.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001133.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000901.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000901.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000901.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001133.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001133.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001133.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001201.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001201.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001264.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001264.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001201.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001201.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001201.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001264.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001264.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001264.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001187.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001187.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000956.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000956.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001187.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001187.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001187.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000956.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000956.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000956.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001361.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001361.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001361.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001361.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001361.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001147.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001147.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001219.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001219.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001147.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001147.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001147.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001219.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001219.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001219.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000817.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000817.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001093.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001093.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001123.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001123.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000817.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000817.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000817.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001093.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001093.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001093.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001123.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001123.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001123.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000924.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000924.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001206.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001206.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000924.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000924.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000924.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001206.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001206.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001206.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000960.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000960.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001177.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001177.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000960.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000960.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000960.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001177.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001177.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001177.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001233.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001233.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001130.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001130.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001335.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001335.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001233.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001233.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001233.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001130.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001130.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001130.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001335.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001335.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001335.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000926.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000926.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001281.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001281.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0000926.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0000926.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000926.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001281.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001281.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001281.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001036.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001036.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001036.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001036.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001036.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001293.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001293.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001293.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001293.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001293.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001350.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001350.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001350.fa -codontable 11\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001350.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001350.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001418.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001418.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n",
      "#------------------------------------------------------------------------#\n",
      "#  Input files:  /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001298.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001298.fna\n",
      "#  Codontable 11 is used\n",
      "#------------------------------------------------------------------------#\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001418.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001418.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001418.fa -codontable 11\n",
      "perl /mnt/maple/ymchen/pal2nal/pal2nal.pl /mnt/maple/ymchen/gene_loss/preflow/msa/OG0001298.fa /mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001298.fna -output fasta > /mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001298.fa -codontable 11\n"
     ]
    }
   ],
   "source": [
    "###now start using Pal2Nal to convert aa alignment into codon-based nucleotide alignment\n",
    "def aa2codon(aa_align,fna_file,target_file):\n",
    "    os.system(f\"perl /mnt/maple/ymchen/pal2nal/pal2nal.pl {aa_align} {fna_file} -output fasta > {target_file} -codontable 11\")\n",
    "    print(f\"perl /mnt/maple/ymchen/pal2nal/pal2nal.pl {aa_align} {fna_file} -output fasta > {target_file} -codontable 11\")\n",
    "def convert_aacodon_all(aa_folder,fna_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(aa_folder):\n",
    "        aa_name=filename.rsplit(\".\",1)[0]\n",
    "        aa_path = os.path.join(aa_folder, aa_name+\".fa\")\n",
    "        for dna_file in os.listdir(fna_folder):\n",
    "            dna_name=dna_file.rsplit(\".\",1)[0]\n",
    "            dna_path = os.path.join(fna_folder, dna_name+\".fna\")\n",
    "            if dna_name==aa_name:\n",
    "                aa2codon(aa_path,dna_path,f\"{target_folder}/{dna_name}.fa\")\n",
    "#aa2codon(\"/mnt/maple/ymchen/gene_loss/preflow/msa/OG0001442.fa\",\"/mnt/maple/ymchen/gene_loss/preflow/correspond_files/OG0001442.fna\",\"/mnt/maple/ymchen/gene_loss/preflow/OG0001442.fa\")                \n",
    "convert_aacodon_all(\"/mnt/maple/ymchen/gene_loss/preflow/msa\",\"/mnt/maple/ymchen/gene_loss/preflow/correspond_files\",\"/mnt/maple/ymchen/gene_loss/preflow/codon_align\")        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "748ef283-9b04-4ab0-b66d-d8ce470ba373",
   "metadata": {},
   "outputs": [],
   "source": [
    "###reivse through trimal\n",
    "def revise_msa_result(align_folder,revise_align_folder):\n",
    "    for filename in os.listdir(align_folder):\n",
    "        filepath = os.path.join(align_folder, filename)\n",
    "        if os.path.isfile(filepath):\n",
    "            os.system(f\"trimal -in {filepath} -out {revise_align_folder}/{filename} -automated1\")\n",
    "revise_msa_result(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align\",\"/mnt/maple/ymchen/gene_loss/preflow/codon_align\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "cc161465-dbf5-4b7a-9161-84cf73c231d7",
   "metadata": {},
   "outputs": [],
   "source": [
    "###convert format of alignment\n",
    "def fas2phy(fas_file,phy_file):\n",
    "    with open(phy_file, \"w\") as phylip_file:\n",
    "        sequences = SeqIO.parse(fas_file, \"fasta\")\n",
    "        SeqIO.write(sequences, phy_file, \"phylip\")\n",
    "#fas2phy(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000803.fa\",\"/mnt/maple/ymchen/gene_loss/preflow/OG0000803.phy\")\n",
    "def convert_fas2phy_all(codon_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(codon_folder):\n",
    "        file_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(codon_folder, filename)\n",
    "        fas2phy(file_path,f\"{target_folder}/{file_name}.phy\")\n",
    "convert_fas2phy_all(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align\",\"/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "1e09ab27-a68a-4762-8c25-b70ac21f40f5",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001351.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001351.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001338.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001338.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001438.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001438.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001394.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001394.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001370.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001370.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001332.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001332.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001200.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001200.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001229.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001229.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001189.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001189.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001356.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001356.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001136.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001136.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001198.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001198.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001392.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001392.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001379.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001379.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001197.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001197.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001381.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001381.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001103.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001103.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001398.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001398.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001108.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001108.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001386.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001386.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001408.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001408.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001045.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001045.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001314.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001314.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001149.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001149.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001359.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001359.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001431.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001431.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001000.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001000.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001271.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001271.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001053.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001053.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001241.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001241.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001249.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001249.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001191.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001191.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001235.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001235.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001070.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001070.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001035.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001035.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001244.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001244.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001151.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001151.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001220.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001220.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000923.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000923.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001396.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001396.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001025.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001025.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001353.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001353.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000935.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000935.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000938.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000938.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001433.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001433.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001135.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001135.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000976.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000976.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001268.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001268.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001100.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001100.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001029.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001029.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001304.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001304.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000876.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000876.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000905.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000905.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001106.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001106.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001329.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001329.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001127.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001127.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000825.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000825.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001279.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001279.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001395.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001395.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001432.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001432.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000847.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000847.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001131.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001131.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001347.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001347.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000807.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000807.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000989.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000989.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001017.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001017.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001321.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001321.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001164.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001164.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001037.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001037.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001165.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001165.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000981.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000981.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001095.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001095.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000822.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000822.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001071.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001071.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001292.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001292.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001160.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001160.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001087.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001087.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001067.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001067.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001344.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001344.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000921.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000921.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001274.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001274.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001099.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001099.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001266.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001266.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001436.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001436.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001009.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001009.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001391.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001391.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001364.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001364.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001243.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001243.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001096.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001096.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001107.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001107.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001010.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001010.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001404.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001404.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001168.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001168.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000803.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000803.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000927.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000927.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000805.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000805.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000993.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000993.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000859.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000859.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000969.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000969.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000912.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000912.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001222.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001222.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000826.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000826.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001173.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001173.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000972.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000972.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001297.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001297.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000880.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000880.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000865.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000865.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001090.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001090.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000990.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000990.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001074.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001074.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000804.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000804.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001066.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001066.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000925.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000925.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000836.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000836.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000857.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000857.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001341.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001341.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000930.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000930.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001311.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001311.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001145.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001145.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001282.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001282.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001119.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001119.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001312.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001312.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001276.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001276.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001144.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001144.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001285.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001285.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000848.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000848.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001199.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001199.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001419.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001419.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001331.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001331.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001410.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001410.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001342.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001342.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000953.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000953.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001259.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001259.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001272.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001272.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001291.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001291.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001002.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001002.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001020.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001020.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001369.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001369.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001286.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001286.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001248.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001248.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000942.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000942.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000855.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000855.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001193.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001193.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000946.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000946.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001415.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001415.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001352.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001352.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001375.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001375.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001358.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001358.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000832.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000832.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000858.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000858.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000862.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000862.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001354.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001354.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001091.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001091.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001429.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001429.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000939.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000939.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001139.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001139.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001016.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001016.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001132.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001132.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001212.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001212.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001218.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001218.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001263.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001263.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001081.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001081.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001031.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001031.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000909.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000909.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001287.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001287.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001337.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001337.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000898.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000898.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000897.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000897.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001348.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001348.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000999.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000999.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001413.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001413.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000831.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000831.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001112.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001112.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001280.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001280.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000823.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000823.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001273.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001273.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000866.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000866.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000919.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000919.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001324.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001324.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001170.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001170.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000982.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000982.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001102.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001102.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000845.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000845.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001365.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001365.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001284.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001284.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001138.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001138.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001308.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001308.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000896.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000896.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001196.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001196.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001004.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001004.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001054.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001054.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001121.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001121.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001376.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001376.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001142.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001142.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000940.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000940.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000885.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000885.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001440.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001440.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000979.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000979.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001072.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001072.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000996.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000996.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000947.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000947.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001134.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001134.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001217.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001217.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000977.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000977.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001377.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001377.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000890.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000890.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000873.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000873.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001178.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001178.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001366.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001366.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001427.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001427.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001320.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001320.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001423.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001423.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001421.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001421.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000929.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000929.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001140.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001140.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000886.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000886.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001385.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001385.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001269.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001269.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001315.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001315.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000871.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000871.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000963.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000963.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000920.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000920.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001042.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001042.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001228.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001228.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000816.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000816.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001209.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001209.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001412.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001412.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001260.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001260.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000833.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000833.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000812.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000812.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001115.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001115.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001378.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001378.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001101.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001101.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001027.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001027.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001428.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001428.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000961.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000961.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001317.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001317.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001011.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001011.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001110.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001110.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000955.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000955.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001182.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001182.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001252.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001252.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001089.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001089.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000899.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000899.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001159.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001159.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001387.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001387.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001283.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001283.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000895.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000895.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000941.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000941.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001063.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001063.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001114.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001114.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001383.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001383.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000893.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000893.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000950.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000950.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001295.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001295.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001299.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001299.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001328.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001328.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001400.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001400.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001414.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001414.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001156.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001156.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000838.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000838.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001215.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001215.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001059.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001059.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001309.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001309.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001007.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001007.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001128.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001128.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001022.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001022.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001319.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001319.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000878.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000878.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001208.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001208.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000902.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000902.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001223.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001223.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001301.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001301.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001254.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001254.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001257.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001257.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001374.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001374.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001109.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001109.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001203.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001203.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000856.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000856.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001049.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001049.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000870.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000870.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001077.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001077.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000984.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000984.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001086.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001086.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001313.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001313.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000915.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000915.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000998.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000998.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000943.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000943.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001240.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001240.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001367.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001367.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001171.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001171.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001401.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001401.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001245.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001245.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000917.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000917.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001092.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001092.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001224.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001224.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001227.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001227.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001380.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001380.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001024.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001024.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001001.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001001.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001175.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001175.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000883.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000883.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001277.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001277.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001270.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001270.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001226.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001226.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001018.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001018.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000975.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000975.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001265.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001265.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000906.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000906.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000931.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000931.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000932.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000932.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001190.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001190.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001399.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001399.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000834.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000834.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001051.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001051.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001343.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001343.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001057.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001057.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001417.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001417.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001261.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001261.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000835.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000835.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000965.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000965.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001294.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001294.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000957.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000957.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001306.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001306.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001120.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001120.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001125.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001125.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001065.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001065.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001064.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001064.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001363.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001363.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001253.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001253.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001094.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001094.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001105.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001105.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000967.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000967.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001050.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001050.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000914.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000914.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000881.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000881.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000951.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000951.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001316.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001316.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001439.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001439.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000872.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000872.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001289.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001289.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000974.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000974.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000937.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000937.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001085.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001085.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001389.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001389.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001288.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001288.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001278.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001278.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000888.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000888.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001043.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001043.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001069.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001069.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000916.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000916.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001172.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001172.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001424.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001424.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001129.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001129.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001137.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001137.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001345.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001345.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001382.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001382.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001111.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001111.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001188.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001188.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001204.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001204.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000851.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000851.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001184.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001184.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000973.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000973.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001056.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001056.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001033.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001033.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001216.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001216.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001355.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001355.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000980.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000980.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001116.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001116.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001310.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001310.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001442.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001442.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001336.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001336.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000806.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000806.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001258.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001258.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000944.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000944.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000849.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000849.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001078.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001078.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001403.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001403.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001055.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001055.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001157.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001157.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000949.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000949.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001434.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001434.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001158.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001158.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000913.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000913.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000867.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000867.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000891.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000891.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000868.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000868.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001058.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001058.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001205.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001205.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001161.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001161.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001422.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001422.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001023.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001023.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001097.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001097.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000869.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000869.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001174.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001174.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001214.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001214.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001118.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001118.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000853.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000853.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001211.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001211.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000837.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000837.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001430.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001430.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001210.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001210.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001192.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001192.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001242.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001242.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001390.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001390.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000966.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000966.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001169.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001169.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001232.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001232.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001409.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001409.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001221.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001221.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001327.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001327.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001126.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001126.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001061.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001061.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000882.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000882.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001238.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001238.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001237.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001237.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000968.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000968.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001225.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001225.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001346.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001346.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001117.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001117.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000995.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000995.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000846.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000846.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000875.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000875.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001323.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001323.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000997.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000997.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001368.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001368.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001339.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001339.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000987.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000987.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001080.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001080.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001019.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001019.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001318.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001318.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000829.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000829.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001305.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001305.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001181.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001181.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001034.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001034.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000962.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000962.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001152.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001152.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000964.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000964.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001062.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001062.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000988.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000988.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001334.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001334.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001307.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001307.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000954.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000954.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001113.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001113.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001186.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001186.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001183.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001183.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000892.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000892.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001425.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001425.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001028.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001028.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001255.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001255.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001076.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001076.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000994.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000994.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001068.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001068.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000922.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000922.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000839.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000839.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001326.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001326.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001207.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001207.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001124.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001124.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000863.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000863.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001008.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001008.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001162.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001162.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001250.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001250.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000874.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000874.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001079.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001079.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001155.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001155.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001239.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001239.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000842.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000842.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001290.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001290.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001179.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001179.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001262.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001262.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001330.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001330.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000908.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000908.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001088.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001088.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001397.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001397.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001373.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001373.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001302.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001302.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001420.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001420.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001437.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001437.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000877.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000877.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001153.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001153.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001402.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001402.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001322.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001322.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001388.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001388.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001333.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001333.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000879.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000879.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001360.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001360.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000860.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000860.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001411.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001411.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001426.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001426.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001122.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001122.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000840.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000840.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001026.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001026.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001384.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001384.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001416.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001416.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001340.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001340.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000901.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000901.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001133.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001133.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001201.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001201.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001264.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001264.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001187.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001187.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000956.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000956.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001361.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001361.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001147.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001147.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001219.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001219.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000817.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000817.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001093.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001093.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001123.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001123.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000924.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000924.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001206.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001206.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000960.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000960.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001177.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001177.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001233.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001233.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001130.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001130.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001335.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001335.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0000926.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0000926.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001281.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001281.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001036.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001036.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001293.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001293.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001350.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001350.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001418.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001418.phy'.\n",
      "FASTA file '/mnt/maple/ymchen/gene_loss/preflow/codon_align/OG0001298.fa' successfully converted to Phylip format as '/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new/OG0001298.phy'.\n"
     ]
    }
   ],
   "source": [
    "###convert (new)\n",
    "def fasta_to_phylip(fasta_file, phylip_file):\n",
    "    sequences = list(SeqIO.parse(fasta_file, \"fasta\"))\n",
    "    seq_count = len(sequences)\n",
    "    seq_length = len(str(sequences[0].seq))\n",
    "    for seq in sequences:\n",
    "        if len(str(seq.seq)) != seq_length:\n",
    "            raise ValueError(f\"All sequences must have the same length. \"\n",
    "                             f\"Sequence {seq.id} has length {len(str(seq.seq))}, expected {seq_length}.\")\n",
    "    with open(phylip_file, 'w') as outfile:\n",
    "        outfile.write(f\"{seq_count} {seq_length}\\n\")\n",
    "        for seq in sequences:\n",
    "            new_name = seq.id[-8:].rjust(10)  \n",
    "            outfile.write(f\"{new_name}  {str(seq.seq)}\\n\")\n",
    "\n",
    "    print(f\"FASTA file '{fasta_file}' successfully converted to Phylip format as '{phylip_file}'.\")\n",
    "def convert_fas_phy_all(codon_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(codon_folder):\n",
    "        file_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(codon_folder, filename)\n",
    "        fasta_to_phylip(file_path,f\"{target_folder}/{file_name}.phy\")\n",
    "convert_fas_phy_all(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align\",\"/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "c18df131-d7e5-4244-b5ef-dc761c66abae",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true,
     "source_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GCGGGGCAT\n",
      "\n",
      "Skipping invalid line:            GGACACGGA\n",
      "\n",
      "Skipping invalid line:            GTGATCCGG\n",
      "\n",
      "Skipping invalid line:            GCGGGCCAT\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            AGGGGTGTG\n",
      "\n",
      "Skipping invalid line:            TCGGGACAT\n",
      "\n",
      "Skipping invalid line:            GCAGGACAT\n",
      "\n",
      "Skipping invalid line:            GGCCACGGA\n",
      "\n",
      "Skipping invalid line:            GGGCACGGC\n",
      "\n",
      "Skipping invalid line:            GCGGGGCAT\n",
      "\n",
      "Skipping invalid line:            GGGGGGCAT\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACAA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACAA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Skipping invalid line:            GTGGGACTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACAA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGGCTA\n",
      "\n",
      "Skipping invalid line:            GTGGGACGA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001081.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000963.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000979.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001009.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000919.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGGGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGGGCAGGA\n",
      "\n",
      "Skipping invalid line:            GGGGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGGGCAGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGGGCAGGA\n",
      "\n",
      "Skipping invalid line:            GGCGCAGGC\n",
      "\n",
      "Skipping invalid line:            GGGGCGGGA\n",
      "\n",
      "Skipping invalid line:            GGCGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGAGCCGGA\n",
      "\n",
      "Skipping invalid line:            GGAGCCGGA\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGCGGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000817.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000916.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001065.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001086.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TCT\n",
      "\n",
      "Skipping invalid line:            TCT\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001117.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCC\n",
      "\n",
      "Skipping invalid line:            GGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGCG\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            GGCG\n",
      "\n",
      "Skipping invalid line:            TGCC\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Skipping invalid line:            GGCT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000968.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Skipping invalid line:            CAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000981.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            CGCCTGAT\n",
      "\n",
      "Skipping invalid line:            CGGTGGAG\n",
      "\n",
      "Skipping invalid line:            CCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCGGAT\n",
      "\n",
      "Skipping invalid line:            CCCCCGAT\n",
      "\n",
      "Skipping invalid line:            CACCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGAC\n",
      "\n",
      "Skipping invalid line:            CCCCTGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGAT\n",
      "\n",
      "Skipping invalid line:            CCCCTGAT\n",
      "\n",
      "Skipping invalid line:            CGCCGGAC\n",
      "\n",
      "Skipping invalid line:            CGCCGGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Skipping invalid line:            TGCCCGAT\n",
      "\n",
      "Skipping invalid line:            TGCCCGAT\n",
      "\n",
      "Skipping invalid line:            TGCCTGAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001158.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001033.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001271.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001342.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001421.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000856.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001341.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001222.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000924.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            GCTA\n",
      "\n",
      "Skipping invalid line:            TCTT\n",
      "\n",
      "Skipping invalid line:            CCTG\n",
      "\n",
      "Skipping invalid line:            GCTG\n",
      "\n",
      "Skipping invalid line:            GATG\n",
      "\n",
      "Skipping invalid line:            ATTG\n",
      "\n",
      "Skipping invalid line:            AATG\n",
      "\n",
      "Skipping invalid line:            CTTG\n",
      "\n",
      "Skipping invalid line:            CCTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ATTG\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Skipping invalid line:            ACTG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001289.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001208.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001269.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCGGTC\n",
      "\n",
      "Skipping invalid line:            CCGGGC\n",
      "\n",
      "Skipping invalid line:            CCCGCC\n",
      "\n",
      "Skipping invalid line:            CCGATC\n",
      "\n",
      "Skipping invalid line:            CGGGGC\n",
      "\n",
      "Skipping invalid line:            CCCACC\n",
      "\n",
      "Skipping invalid line:            CCGGGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAGCC\n",
      "\n",
      "Skipping invalid line:            CCAAAG\n",
      "\n",
      "Skipping invalid line:            CCGGCC\n",
      "\n",
      "Skipping invalid line:            CCGAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAGGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAGGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAGGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Skipping invalid line:            CCGAGC\n",
      "\n",
      "Skipping invalid line:            CCGAGC\n",
      "\n",
      "Skipping invalid line:            CCAAGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000906.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ACCCGT\n",
      "\n",
      "Skipping invalid line:            ACCCGT\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CATTCC\n",
      "\n",
      "Skipping invalid line:            CACGAT\n",
      "\n",
      "Skipping invalid line:            CATCCA\n",
      "\n",
      "Skipping invalid line:            CACGCC\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACGGG\n",
      "\n",
      "Skipping invalid line:            CATCCG\n",
      "\n",
      "Skipping invalid line:            CACCCC\n",
      "\n",
      "Skipping invalid line:            CATCCA\n",
      "\n",
      "Skipping invalid line:            CACGAC\n",
      "\n",
      "Skipping invalid line:            CACTCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Skipping invalid line:            CACCCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001020.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GACGCGGCGG \n",
      "\n",
      "Skipping invalid line:            GACACGCCGC \n",
      "\n",
      "Skipping invalid line:            GACCCGGCGC \n",
      "\n",
      "Skipping invalid line:            GCCGCGCCAA \n",
      "\n",
      "Skipping invalid line:            GACGCGGCGG \n",
      "\n",
      "Skipping invalid line:            GACGCGACGC \n",
      "\n",
      "Skipping invalid line:            GGCGCGGCGC \n",
      "\n",
      "Skipping invalid line:            GACGCGGCGC \n",
      "\n",
      "Skipping invalid line:            GACACGGCGC \n",
      "\n",
      "Skipping invalid line:            GACGCGCCGT \n",
      "\n",
      "Skipping invalid line:            GACCCGCCGG \n",
      "\n",
      "Skipping invalid line:            GAACAAGCTC \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGA \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGA \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGA \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCAAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAACGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAACGG \n",
      "\n",
      "Skipping invalid line:            GCCGAAGCGG \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000870.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001378.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001093.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001056.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001260.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            CCA\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            ACA\n",
      "\n",
      "Skipping invalid line:            CCA\n",
      "\n",
      "Skipping invalid line:            ACA\n",
      "\n",
      "Skipping invalid line:            TCA\n",
      "\n",
      "Skipping invalid line:            ACA\n",
      "\n",
      "Skipping invalid line:            TCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            ACA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            ACA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001423.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000859.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001054.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001142.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001016.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGATGG\n",
      "\n",
      "Skipping invalid line:            AGATGG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            AGTGGC\n",
      "\n",
      "Skipping invalid line:            TTCGAC\n",
      "\n",
      "Skipping invalid line:            GAGACC\n",
      "\n",
      "Skipping invalid line:            GGCGAG\n",
      "\n",
      "Skipping invalid line:            GGTGAG\n",
      "\n",
      "Skipping invalid line:            GGTGGC\n",
      "\n",
      "Skipping invalid line:            GGTAGG\n",
      "\n",
      "Skipping invalid line:            GGTGCG\n",
      "\n",
      "Skipping invalid line:            GGGGCC\n",
      "\n",
      "Skipping invalid line:            GCATCA\n",
      "\n",
      "Skipping invalid line:            GATGCG\n",
      "\n",
      "Skipping invalid line:            TTTGGA\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Skipping invalid line:            GAAGCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001206.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001318.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001064.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001287.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GGAAGGGG\n",
      "\n",
      "Skipping invalid line:            GGCAGAGT\n",
      "\n",
      "Skipping invalid line:            GCCAGCGC\n",
      "\n",
      "Skipping invalid line:            GCAAGCGC\n",
      "\n",
      "Skipping invalid line:            GGCAGGGC\n",
      "\n",
      "Skipping invalid line:            GCAAGGGC\n",
      "\n",
      "Skipping invalid line:            GACAAGGA\n",
      "\n",
      "Skipping invalid line:            GCAAGGGC\n",
      "\n",
      "Skipping invalid line:            GCGGCAAA\n",
      "\n",
      "Skipping invalid line:            GGGGTGGT\n",
      "\n",
      "Skipping invalid line:            GGCAATCG\n",
      "\n",
      "Skipping invalid line:            GCGAGCAG\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAGCCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Skipping invalid line:            GAAACCGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001105.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001120.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001145.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000962.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001299.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000858.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000917.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001091.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001380.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001330.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000977.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001210.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            TCGGG\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            AGGGC\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            ACAGC\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000901.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000838.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000888.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001335.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001288.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000860.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001317.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001408.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001031.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001295.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000879.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000930.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TT\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TG\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TG\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Skipping invalid line:            TG\n",
      "\n",
      "Skipping invalid line:            TG\n",
      "\n",
      "Skipping invalid line:            TC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001248.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001178.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001182.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000915.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000902.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000955.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000995.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TCCGCGGT\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TCCGGGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGGGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGTC\n",
      "\n",
      "Skipping invalid line:            TCCGGGGC\n",
      "\n",
      "Skipping invalid line:            TCCGCGGG\n",
      "\n",
      "Skipping invalid line:            TCAAGGGA\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGCC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Skipping invalid line:            TTCGCGTT\n",
      "\n",
      "Skipping invalid line:            TTCGCGTT\n",
      "\n",
      "Skipping invalid line:            TTCGCGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000833.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GGGCG\n",
      "\n",
      "Skipping invalid line:            GCCGG\n",
      "\n",
      "Skipping invalid line:            GCGCG\n",
      "\n",
      "Skipping invalid line:            GGGCG\n",
      "\n",
      "Skipping invalid line:            GGGCC\n",
      "\n",
      "Skipping invalid line:            GGGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GGGCC\n",
      "\n",
      "Skipping invalid line:            GCAAG\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GGCTG\n",
      "\n",
      "Skipping invalid line:            GGCGC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCT\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCT\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Skipping invalid line:            GCGCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001103.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000895.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001122.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001129.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001164.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            CCGACGGCT\n",
      "\n",
      "Skipping invalid line:            CCCCAAGCC\n",
      "\n",
      "Skipping invalid line:            CCCGCCGGC\n",
      "\n",
      "Skipping invalid line:            CCGGCGGCC\n",
      "\n",
      "Skipping invalid line:            CCCGCCGAG\n",
      "\n",
      "Skipping invalid line:            GGCGCGGCC\n",
      "\n",
      "Skipping invalid line:            CCCGCC---\n",
      "\n",
      "Skipping invalid line:            GAGGCCACC\n",
      "\n",
      "Skipping invalid line:            CCCAAGGCG\n",
      "\n",
      "Skipping invalid line:            CCGAAGGAC\n",
      "\n",
      "Skipping invalid line:            CCCAAGGCG\n",
      "\n",
      "Skipping invalid line:            ACCGGAAGC\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCC\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTCTG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCG\n",
      "\n",
      "Skipping invalid line:            GAGGGTGCT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000912.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001237.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            GCTGCCAG\n",
      "\n",
      "Skipping invalid line:            CCTGCCCA\n",
      "\n",
      "Skipping invalid line:            GGTGTTTG\n",
      "\n",
      "Skipping invalid line:            GCTGCCAG\n",
      "\n",
      "Skipping invalid line:            GCTGCAAT\n",
      "\n",
      "Skipping invalid line:            GGTGTTCG\n",
      "\n",
      "Skipping invalid line:            GCTGCAAT\n",
      "\n",
      "Skipping invalid line:            ACTGCACG\n",
      "\n",
      "Skipping invalid line:            GCTGCGCA\n",
      "\n",
      "Skipping invalid line:            GCTGCTCG\n",
      "\n",
      "Skipping invalid line:            CCTGCCCC\n",
      "\n",
      "Skipping invalid line:            GGTGCTCT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Skipping invalid line:            CGTGCAAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001070.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001137.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001197.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001113.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001017.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001004.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001087.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001375.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000849.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GCCAA\n",
      "\n",
      "Skipping invalid line:            GGGGA\n",
      "\n",
      "Skipping invalid line:            GGCAC\n",
      "\n",
      "Skipping invalid line:            GTAAC\n",
      "\n",
      "Skipping invalid line:            GACAC\n",
      "\n",
      "Skipping invalid line:            GTCAC\n",
      "\n",
      "Skipping invalid line:            GGCAC\n",
      "\n",
      "Skipping invalid line:            GGCAA\n",
      "\n",
      "Skipping invalid line:            GTTAT\n",
      "\n",
      "Skipping invalid line:            GGCGC\n",
      "\n",
      "Skipping invalid line:            GACGG\n",
      "\n",
      "Skipping invalid line:            GCGGA\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Skipping invalid line:            GGGGG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000998.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            TACC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            T---\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            CCGG\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GAAA\n",
      "\n",
      "Skipping invalid line:            TTAC\n",
      "\n",
      "Skipping invalid line:            GACC\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GCGC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATTC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            AGCC\n",
      "\n",
      "Skipping invalid line:            AGCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            AGCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            AGCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATTC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATTC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATTC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Skipping invalid line:            ATCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001058.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001440.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001283.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001385.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001368.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001351.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001132.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GGACC\n",
      "\n",
      "Skipping invalid line:            GCACG\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GTACC\n",
      "\n",
      "Skipping invalid line:            GGACA\n",
      "\n",
      "Skipping invalid line:            GGACC\n",
      "\n",
      "Skipping invalid line:            GAACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Skipping invalid line:            GCACC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000832.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001343.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            GTTTGCC\n",
      "\n",
      "Skipping invalid line:            GCGAGGC\n",
      "\n",
      "Skipping invalid line:            CACCGCC\n",
      "\n",
      "Skipping invalid line:            GTTTGCG\n",
      "\n",
      "Skipping invalid line:            -CAGAAT\n",
      "\n",
      "Skipping invalid line:            -GCAACC\n",
      "\n",
      "Skipping invalid line:            -CAGAAC\n",
      "\n",
      "Skipping invalid line:            -GCCTCC\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            CATAGGA\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            -CCGACC\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Skipping invalid line:            AGGAGTA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000921.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001152.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001133.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001218.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000845.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001411.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCA\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            AAC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001097.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001079.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000954.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001057.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ACCCAG\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCCGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCCGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000816.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001327.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001277.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001396.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001242.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001319.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001226.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001353.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGCAGCCGCA \n",
      "\n",
      "Skipping invalid line:            CCGCAGCCGC \n",
      "\n",
      "Skipping invalid line:            GCCGCAACCT \n",
      "\n",
      "Skipping invalid line:            GCCGCAACCT \n",
      "\n",
      "Skipping invalid line:            GGGCCTC--- \n",
      "\n",
      "Skipping invalid line:            GCCCACCCCG \n",
      "\n",
      "Skipping invalid line:            GCAG------ \n",
      "\n",
      "Skipping invalid line:            GGGCCTC--- \n",
      "\n",
      "Skipping invalid line:            AGGCAGCGCA \n",
      "\n",
      "Skipping invalid line:            GGAAGCA--- \n",
      "\n",
      "Skipping invalid line:            GGAAATCGGC \n",
      "\n",
      "Skipping invalid line:            AGGTCTGGTG \n",
      "\n",
      "Skipping invalid line:            CCCGGAATCG \n",
      "\n",
      "Skipping invalid line:            TCCTGCCCTG \n",
      "\n",
      "Skipping invalid line:            CCCGGCCCGT \n",
      "\n",
      "Skipping invalid line:            AAGTGAA--- \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCACAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCACAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCACAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAACCT \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCTGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCTGCCG \n",
      "\n",
      "Skipping invalid line:            GCCGCAACCT \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001286.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            CG\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            CG\n",
      "\n",
      "Skipping invalid line:            CT\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001243.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001138.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001418.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001214.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001189.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000953.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            ATC---\n",
      "\n",
      "Skipping invalid line:            GCGGTT\n",
      "\n",
      "Skipping invalid line:            GCCGCC\n",
      "\n",
      "Skipping invalid line:            GTGTTC\n",
      "\n",
      "Skipping invalid line:            GCGGCC\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            AGTCGA\n",
      "\n",
      "Skipping invalid line:            GCGGCA\n",
      "\n",
      "Skipping invalid line:            CGGCGG\n",
      "\n",
      "Skipping invalid line:            CGCTTC\n",
      "\n",
      "Skipping invalid line:            GGCCTG\n",
      "\n",
      "Skipping invalid line:            CGGGCC\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCA\n",
      "\n",
      "Skipping invalid line:            GCGGCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001404.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AGC\n",
      "\n",
      "Skipping invalid line:            CCC\n",
      "\n",
      "Skipping invalid line:            GAA\n",
      "\n",
      "Skipping invalid line:            AGC\n",
      "\n",
      "Skipping invalid line:            CGC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            CCC\n",
      "\n",
      "Skipping invalid line:            GAG\n",
      "\n",
      "Skipping invalid line:            CGT\n",
      "\n",
      "Skipping invalid line:            CCA\n",
      "\n",
      "Skipping invalid line:            CGC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001316.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000967.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000943.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AGCGGGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCG\n",
      "\n",
      "Skipping invalid line:            AGCGGGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCA\n",
      "\n",
      "Skipping invalid line:            AGCGCGCA\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGGGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Skipping invalid line:            AACGCGCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001409.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001274.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGA\n",
      "\n",
      "Skipping invalid line:            GGAA\n",
      "\n",
      "Skipping invalid line:            CGGA\n",
      "\n",
      "Skipping invalid line:            CGGA\n",
      "\n",
      "Skipping invalid line:            AGGG\n",
      "\n",
      "Skipping invalid line:            CGGA\n",
      "\n",
      "Skipping invalid line:            CGGG\n",
      "\n",
      "Skipping invalid line:            CGGA\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            TGGA\n",
      "\n",
      "Skipping invalid line:            CGCG\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            TGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001199.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001227.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001108.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGGGCC\n",
      "\n",
      "Skipping invalid line:            TCCGGG\n",
      "\n",
      "Skipping invalid line:            GCA---\n",
      "\n",
      "Skipping invalid line:            CGCAGC\n",
      "\n",
      "Skipping invalid line:            CGGGGC\n",
      "\n",
      "Skipping invalid line:            CGGGGC\n",
      "\n",
      "Skipping invalid line:            CGCGGT\n",
      "\n",
      "Skipping invalid line:            CGA---\n",
      "\n",
      "Skipping invalid line:            CAACCC\n",
      "\n",
      "Skipping invalid line:            GGAGAT\n",
      "\n",
      "Skipping invalid line:            GGCGCC\n",
      "\n",
      "Skipping invalid line:            CGG---\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Skipping invalid line:            CGCGGT\n",
      "\n",
      "Skipping invalid line:            CGCGGT\n",
      "\n",
      "Skipping invalid line:            CGTGGT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001373.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001399.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001391.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001326.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000993.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000974.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001293.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001369.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001442.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001109.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001323.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001085.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000837.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001281.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001307.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000834.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GACCGGTCC\n",
      "\n",
      "Skipping invalid line:            GACCGGGCA\n",
      "\n",
      "Skipping invalid line:            GACCGGTCT\n",
      "\n",
      "Skipping invalid line:            GACCGGTCC\n",
      "\n",
      "Skipping invalid line:            GACCAGGCA\n",
      "\n",
      "Skipping invalid line:            GACCGCGGG\n",
      "\n",
      "Skipping invalid line:            GACCAGGCA\n",
      "\n",
      "Skipping invalid line:            GATCGCAGC\n",
      "\n",
      "Skipping invalid line:            GGCCCCGGA\n",
      "\n",
      "Skipping invalid line:            GACGCGAAC\n",
      "\n",
      "Skipping invalid line:            GGCTCCGGA\n",
      "\n",
      "Skipping invalid line:            GACCGGAGT\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Skipping invalid line:            GATCGTGCA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001207.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001107.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGCGTC\n",
      "\n",
      "Skipping invalid line:            GCGGCGCG\n",
      "\n",
      "Skipping invalid line:            GCGGGGTG\n",
      "\n",
      "Skipping invalid line:            GGGGCCTC\n",
      "\n",
      "Skipping invalid line:            GCGGACTG\n",
      "\n",
      "Skipping invalid line:            GCGGCGTC\n",
      "\n",
      "Skipping invalid line:            GCGGTCTG\n",
      "\n",
      "Skipping invalid line:            GCGGGGTG\n",
      "\n",
      "Skipping invalid line:            GCGGCGCG\n",
      "\n",
      "Skipping invalid line:            GTGGCAGC\n",
      "\n",
      "Skipping invalid line:            GCGGGAAT\n",
      "\n",
      "Skipping invalid line:            GCGGCGAC\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTGCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000842.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TTACA\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TCTAT\n",
      "\n",
      "Skipping invalid line:            TGTAC\n",
      "\n",
      "Skipping invalid line:            TCTAT\n",
      "\n",
      "Skipping invalid line:            TGTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TCTAC\n",
      "\n",
      "Skipping invalid line:            TCTAT\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Skipping invalid line:            TTTAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001114.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001221.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000914.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001026.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001074.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001302.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001430.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001102.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001168.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001089.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001363.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001116.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000840.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001170.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001068.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001035.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001410.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001080.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            CGGATTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            GGGCTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGTTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Skipping invalid line:            AGGGTTCATG \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000931.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000848.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000944.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000925.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001333.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001374.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001358.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Skipping invalid line:            TGG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001115.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001225.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001312.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000877.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000892.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001417.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000880.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001426.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001019.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001010.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001301.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001395.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001328.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001008.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001252.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001238.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001280.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001419.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001282.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001354.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000897.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATTACC\n",
      "\n",
      "Skipping invalid line:            ATGACC\n",
      "\n",
      "Skipping invalid line:            ATCAAG\n",
      "\n",
      "Skipping invalid line:            ATCACT\n",
      "\n",
      "Skipping invalid line:            ATCGGA\n",
      "\n",
      "Skipping invalid line:            ATCTCG\n",
      "\n",
      "Skipping invalid line:            ATCAGT\n",
      "\n",
      "Skipping invalid line:            ATTACG\n",
      "\n",
      "Skipping invalid line:            ---CGT\n",
      "\n",
      "Skipping invalid line:            ---GCG\n",
      "\n",
      "Skipping invalid line:            ---CGT\n",
      "\n",
      "Skipping invalid line:            ATCTCC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Skipping invalid line:            ATCAGC\n",
      "\n",
      "Skipping invalid line:            ATCAGC\n",
      "\n",
      "Skipping invalid line:            ATCGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001209.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001324.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001069.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001235.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCACCCTA\n",
      "\n",
      "Skipping invalid line:            CTCCATGCC\n",
      "\n",
      "Skipping invalid line:            GTGGAGGCC\n",
      "\n",
      "Skipping invalid line:            GCCCACATA\n",
      "\n",
      "Skipping invalid line:            CTGGACGAG\n",
      "\n",
      "Skipping invalid line:            GTGGCGCTG\n",
      "\n",
      "Skipping invalid line:            CTGGACGAG\n",
      "\n",
      "Skipping invalid line:            GTGGTGGCA\n",
      "\n",
      "Skipping invalid line:            GCGTCG---\n",
      "\n",
      "Skipping invalid line:            GAACAGGCG\n",
      "\n",
      "Skipping invalid line:            GCGCATGTC\n",
      "\n",
      "Skipping invalid line:            GCACATGTC\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Skipping invalid line:            GCCCATGCT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001011.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGTTT\n",
      "\n",
      "Skipping invalid line:            CCGGGTTC\n",
      "\n",
      "Skipping invalid line:            CCGGCTTT\n",
      "\n",
      "Skipping invalid line:            TTGGATTT\n",
      "\n",
      "Skipping invalid line:            CGGGGTTT\n",
      "\n",
      "Skipping invalid line:            CAGGGTTC\n",
      "\n",
      "Skipping invalid line:            CCGGGTTC\n",
      "\n",
      "Skipping invalid line:            CGGGATTT\n",
      "\n",
      "Skipping invalid line:            CGGGCTTT\n",
      "\n",
      "Skipping invalid line:            CGGGATTT\n",
      "\n",
      "Skipping invalid line:            CAGGATTT\n",
      "\n",
      "Skipping invalid line:            CAGGATAT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Skipping invalid line:            CAGGGCTT\n",
      "\n",
      "Skipping invalid line:            CAGGGCTT\n",
      "\n",
      "Skipping invalid line:            CGGGGCTT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001263.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001309.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000873.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000956.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001412.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001029.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000853.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001424.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001394.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000886.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001027.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000926.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CCA\n",
      "\n",
      "Skipping invalid line:            ---\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            AGT\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAG\n",
      "\n",
      "Skipping invalid line:            CAG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            CGA\n",
      "\n",
      "Skipping invalid line:            TCG\n",
      "\n",
      "Skipping invalid line:            CTG\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Skipping invalid line:            CAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000874.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001420.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001066.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000973.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000932.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAC\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACGCAC\n",
      "\n",
      "Skipping invalid line:            TGGACTCAC\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACCCAC\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Skipping invalid line:            TGGACGCAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001361.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001439.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000829.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001119.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001151.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000950.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001188.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000939.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001131.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ---ACG\n",
      "\n",
      "Skipping invalid line:            ---TCC\n",
      "\n",
      "Skipping invalid line:            ---GCA\n",
      "\n",
      "Skipping invalid line:            ---CGC\n",
      "\n",
      "Skipping invalid line:            CCCGAT\n",
      "\n",
      "Skipping invalid line:            GAAGTC\n",
      "\n",
      "Skipping invalid line:            GCGGAT\n",
      "\n",
      "Skipping invalid line:            GACGCT\n",
      "\n",
      "Skipping invalid line:            CAGGTC\n",
      "\n",
      "Skipping invalid line:            ACACGC\n",
      "\n",
      "Skipping invalid line:            TCGGCC\n",
      "\n",
      "Skipping invalid line:            GCCGGC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCAC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCAC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCAC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Skipping invalid line:            ATGCCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000882.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000957.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001258.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Skipping invalid line:            GAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001162.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001329.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            GCAAG\n",
      "\n",
      "Skipping invalid line:            CGAAG\n",
      "\n",
      "Skipping invalid line:            CCAAA\n",
      "\n",
      "Skipping invalid line:            CCAAG\n",
      "\n",
      "Skipping invalid line:            CGAAA\n",
      "\n",
      "Skipping invalid line:            CCAGG\n",
      "\n",
      "Skipping invalid line:            CGAAG\n",
      "\n",
      "Skipping invalid line:            CCAAA\n",
      "\n",
      "Skipping invalid line:            GTCGC\n",
      "\n",
      "Skipping invalid line:            CCCGC\n",
      "\n",
      "Skipping invalid line:            CCCGC\n",
      "\n",
      "Skipping invalid line:            AGAAG\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Skipping invalid line:            CAAAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000822.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000804.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001431.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001241.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000893.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GTCA\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            CCCG\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            AGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Skipping invalid line:            GGGG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001100.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000937.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001240.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TTC\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TAT\n",
      "\n",
      "Skipping invalid line:            TTC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Skipping invalid line:            TAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001346.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001042.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001337.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001438.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001306.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            GGCGAGG--- \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            GGCCGCA--- \n",
      "\n",
      "Skipping invalid line:            GGCG------ \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            GGCGCTTCGG \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            GGCCGCGCGC \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            ---------- \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCGG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCGG \n",
      "\n",
      "Skipping invalid line:            AAGCTATCCG \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001278.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            GTT\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            GTC\n",
      "\n",
      "Skipping invalid line:            ATC\n",
      "\n",
      "Skipping invalid line:            GTA\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            TCC\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            ATG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Skipping invalid line:            GTG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001268.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001045.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000923.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001050.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001092.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000805.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001344.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000984.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001383.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001285.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001144.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001123.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001177.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001090.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001379.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001192.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000899.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000875.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001106.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001345.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001018.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001321.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001036.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000927.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001155.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001350.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001124.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001272.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001025.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001229.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000996.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001135.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000857.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001053.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001334.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001059.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            AATGCCGCGC \n",
      "\n",
      "Skipping invalid line:            GATGCCCCAA \n",
      "\n",
      "Skipping invalid line:            GATGCCACGG \n",
      "\n",
      "Skipping invalid line:            CATGCCGCGC \n",
      "\n",
      "Skipping invalid line:            GATGCCGCGC \n",
      "\n",
      "Skipping invalid line:            CGTGGCGCAC \n",
      "\n",
      "Skipping invalid line:            CATGCCCCGC \n",
      "\n",
      "Skipping invalid line:            GATGCCCCGT \n",
      "\n",
      "Skipping invalid line:            GATGCCTCGC \n",
      "\n",
      "Skipping invalid line:            GATGCCGCGG \n",
      "\n",
      "Skipping invalid line:            GATGCCGAGA \n",
      "\n",
      "Skipping invalid line:            GATGCCGCGC \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTGAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTGAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTGAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTGAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Skipping invalid line:            GCTTAAGCGG \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001071.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000863.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000855.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001290.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000823.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001367.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001377.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000975.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000941.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001265.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000826.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000964.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001223.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            CTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            CTTCGCGGAC \n",
      "\n",
      "Skipping invalid line:            GTTCGCCGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCCGAC \n",
      "\n",
      "Skipping invalid line:            GTTTGCCGAC \n",
      "\n",
      "Skipping invalid line:            GTTCGTGGAC \n",
      "\n",
      "Skipping invalid line:            GTTTGCCGAC \n",
      "\n",
      "Skipping invalid line:            CTTCGCAGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGAGGAT \n",
      "\n",
      "Skipping invalid line:            CTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            ATTCGAGGAC \n",
      "\n",
      "Skipping invalid line:            GTTCGCCGAC \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Skipping invalid line:            GTTCGCGGAT \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001037.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001203.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000872.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001415.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000867.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000869.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001253.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CTTGCC\n",
      "\n",
      "Skipping invalid line:            GCGCCC\n",
      "\n",
      "Skipping invalid line:            CTTGCC\n",
      "\n",
      "Skipping invalid line:            CTTGCC\n",
      "\n",
      "Skipping invalid line:            CGACAC\n",
      "\n",
      "Skipping invalid line:            CTGCCC\n",
      "\n",
      "Skipping invalid line:            CGCGCA\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            AGATCC\n",
      "\n",
      "Skipping invalid line:            AAGGAC\n",
      "\n",
      "Skipping invalid line:            CGAGAG\n",
      "\n",
      "Skipping invalid line:            CTGGCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Skipping invalid line:            CGACCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001156.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001270.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000987.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001043.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001245.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001002.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCCGCC\n",
      "\n",
      "Skipping invalid line:            AAAGCC\n",
      "\n",
      "Skipping invalid line:            AGCGCC\n",
      "\n",
      "Skipping invalid line:            GCTGCG\n",
      "\n",
      "Skipping invalid line:            AAGGCG\n",
      "\n",
      "Skipping invalid line:            CGCGCG\n",
      "\n",
      "Skipping invalid line:            AAGGCG\n",
      "\n",
      "Skipping invalid line:            CGCGCT\n",
      "\n",
      "Skipping invalid line:            TCGGTG\n",
      "\n",
      "Skipping invalid line:            AAAGCG\n",
      "\n",
      "Skipping invalid line:            GCGGTT\n",
      "\n",
      "Skipping invalid line:            AAGGTT\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Skipping invalid line:            GCAGCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000913.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000966.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000946.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            TTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGG\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            TTGCGG\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGT\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGG\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGG\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTACGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTACGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTACGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Skipping invalid line:            CTGCGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000831.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGAGCAA\n",
      "\n",
      "Skipping invalid line:            GGAGACT\n",
      "\n",
      "Skipping invalid line:            GGAGAGC\n",
      "\n",
      "Skipping invalid line:            GGAGCGC\n",
      "\n",
      "Skipping invalid line:            AGAGGGA\n",
      "\n",
      "Skipping invalid line:            GGATGGC\n",
      "\n",
      "Skipping invalid line:            CGAGGGA\n",
      "\n",
      "Skipping invalid line:            GGACGGC\n",
      "\n",
      "Skipping invalid line:            AGAAGGC\n",
      "\n",
      "Skipping invalid line:            GGAGCCC\n",
      "\n",
      "Skipping invalid line:            AGAGGAC\n",
      "\n",
      "Skipping invalid line:            GGACAGC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATAGC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATAGC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATAGC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Skipping invalid line:            GGATACC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001159.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001249.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001149.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000881.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001130.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001390.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001359.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            GCAAGATG\n",
      "\n",
      "Skipping invalid line:            CCGGACTG\n",
      "\n",
      "Skipping invalid line:            --------\n",
      "\n",
      "Skipping invalid line:            GCAAGATG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTT\n",
      "\n",
      "Skipping invalid line:            GCAAGATG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            GCCGTATG\n",
      "\n",
      "Skipping invalid line:            CGAAAATG\n",
      "\n",
      "Skipping invalid line:            CCAAAATG\n",
      "\n",
      "Skipping invalid line:            CGAAGATG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTC\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Skipping invalid line:            CCAAGCTC\n",
      "\n",
      "Skipping invalid line:            CCAAGCTG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001401.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Skipping invalid line:            CC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001320.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000920.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001228.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            -ATATCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            TGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCC\n",
      "\n",
      "Skipping invalid line:            GGCGGCG\n",
      "\n",
      "Skipping invalid line:            CGCCGCG\n",
      "\n",
      "Skipping invalid line:            GGCGGCC\n",
      "\n",
      "Skipping invalid line:            GGCCGCG\n",
      "\n",
      "Skipping invalid line:            GGCCGCG\n",
      "\n",
      "Skipping invalid line:            GGCCGCG\n",
      "\n",
      "Skipping invalid line:            TGCCGCC\n",
      "\n",
      "Skipping invalid line:            GGCCGCT\n",
      "\n",
      "Skipping invalid line:            GGCCGCC\n",
      "\n",
      "Skipping invalid line:            GGCCGCA\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCAGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCAGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCAGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCAGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCC\n",
      "\n",
      "Skipping invalid line:            AGCAGCT\n",
      "\n",
      "Skipping invalid line:            AGCAGCT\n",
      "\n",
      "Skipping invalid line:            GGCGGCT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001264.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001055.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001172.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000908.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001049.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001432.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000909.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CTCT\n",
      "\n",
      "Skipping invalid line:            CTCT\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            CCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            TCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GATG\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GATG\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTT\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Skipping invalid line:            GCTC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001313.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            GAG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            AAA\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            GCG\n",
      "\n",
      "Skipping invalid line:            ---\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Skipping invalid line:            GCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001414.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000862.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000835.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001095.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001298.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001125.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001217.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001262.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001134.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001244.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000878.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001403.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001389.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            ----\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GCGT\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GCCT\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GCAG\n",
      "\n",
      "Skipping invalid line:            GCAC\n",
      "\n",
      "Skipping invalid line:            GGAG\n",
      "\n",
      "Skipping invalid line:            GTCC\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Skipping invalid line:            GAAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001292.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000990.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            CAGGCAG\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            GGCGGTG\n",
      "\n",
      "Skipping invalid line:            CGCCGTC\n",
      "\n",
      "Skipping invalid line:            GTCCGCC\n",
      "\n",
      "Skipping invalid line:            GTCGGTG\n",
      "\n",
      "Skipping invalid line:            GACCGCC\n",
      "\n",
      "Skipping invalid line:            GGCGGCG\n",
      "\n",
      "Skipping invalid line:            CAAAGCC\n",
      "\n",
      "Skipping invalid line:            AAGCGCG\n",
      "\n",
      "Skipping invalid line:            TGCCGCC\n",
      "\n",
      "Skipping invalid line:            GAAGGCT\n",
      "\n",
      "Skipping invalid line:            GGCCGCC\n",
      "\n",
      "Skipping invalid line:            CGCCGCC\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Skipping invalid line:            ACAGTCA\n",
      "\n",
      "Skipping invalid line:            ACAGTCA\n",
      "\n",
      "Skipping invalid line:            ACAGGCA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000871.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001216.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCCCTG\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTT\n",
      "\n",
      "Skipping invalid line:            CGCCCTG\n",
      "\n",
      "Skipping invalid line:            CGCTCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTG\n",
      "\n",
      "Skipping invalid line:            CGCACTT\n",
      "\n",
      "Skipping invalid line:            TGCCGCG\n",
      "\n",
      "Skipping invalid line:            TGCATTG\n",
      "\n",
      "Skipping invalid line:            TGCCGCC\n",
      "\n",
      "Skipping invalid line:            CGCCCTG\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            TGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            TGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            TGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Skipping invalid line:            CGCTCTC\n",
      "\n",
      "Skipping invalid line:            CGCTCTC\n",
      "\n",
      "Skipping invalid line:            CGCGCTC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001186.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ACGCG\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ACGGT\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ACGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGT\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGT\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Skipping invalid line:            ATGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001332.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCCCCGT\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCACAA\n",
      "\n",
      "Skipping invalid line:            GCCGCAT\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            GCCGCAG\n",
      "\n",
      "Skipping invalid line:            GCCACGC\n",
      "\n",
      "Skipping invalid line:            GCCGCAG\n",
      "\n",
      "Skipping invalid line:            GCCCCGT\n",
      "\n",
      "Skipping invalid line:            TCCGCTG\n",
      "\n",
      "Skipping invalid line:            CCCCGAG\n",
      "\n",
      "Skipping invalid line:            GCCGAAA\n",
      "\n",
      "Skipping invalid line:            TCCCGAA\n",
      "\n",
      "Skipping invalid line:            GCCGCAA\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCA\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Skipping invalid line:            GCCCCCG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001001.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            TCCGGG\n",
      "\n",
      "Skipping invalid line:            TCTCGG\n",
      "\n",
      "Skipping invalid line:            CAGCGG\n",
      "\n",
      "Skipping invalid line:            CCCAAG\n",
      "\n",
      "Skipping invalid line:            AGC---\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            ------\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAACCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Skipping invalid line:            ACGCCT\n",
      "\n",
      "Skipping invalid line:            ACGCCT\n",
      "\n",
      "Skipping invalid line:            AAATCT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001304.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001187.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            GAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CGA\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            AAG\n",
      "\n",
      "Skipping invalid line:            CCC\n",
      "\n",
      "Skipping invalid line:            CCC\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Skipping invalid line:            CAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001200.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001067.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001219.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGAGCGCC\n",
      "\n",
      "Skipping invalid line:            TGGGGTCGC\n",
      "\n",
      "Skipping invalid line:            TGGGGCGCC\n",
      "\n",
      "Skipping invalid line:            TGGGGTAGC\n",
      "\n",
      "Skipping invalid line:            TGGGGTGCG\n",
      "\n",
      "Skipping invalid line:            TGGGGTGCG\n",
      "\n",
      "Skipping invalid line:            TGGGCCTCT\n",
      "\n",
      "Skipping invalid line:            TGGGGCGAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCGG\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAG\n",
      "\n",
      "Skipping invalid line:            TGGGGCTCC\n",
      "\n",
      "Skipping invalid line:            TGGGGCAAG\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGAGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGAGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGAGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Skipping invalid line:            TGGGGCCAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001190.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001291.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            CTTCGCC\n",
      "\n",
      "Skipping invalid line:            ACTGGGC\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            CTTCGCG\n",
      "\n",
      "Skipping invalid line:            GCTGGGC\n",
      "\n",
      "Skipping invalid line:            TTTCGGC\n",
      "\n",
      "Skipping invalid line:            GCTCGGC\n",
      "\n",
      "Skipping invalid line:            TTTCGCG\n",
      "\n",
      "Skipping invalid line:            CTTCGCC\n",
      "\n",
      "Skipping invalid line:            TTTCGCT\n",
      "\n",
      "Skipping invalid line:            GCTCGGA\n",
      "\n",
      "Skipping invalid line:            GGTGGCC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GCTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            TTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Skipping invalid line:            GTTTGAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000866.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000994.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001338.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001161.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001259.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001297.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000951.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001110.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001201.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ACCCCT\n",
      "\n",
      "Skipping invalid line:            GCCCCC\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGGAA\n",
      "\n",
      "Skipping invalid line:            GCGGAA\n",
      "\n",
      "Skipping invalid line:            GCCGAA\n",
      "\n",
      "Skipping invalid line:            GCCGAA\n",
      "\n",
      "Skipping invalid line:            GCAGAG\n",
      "\n",
      "Skipping invalid line:            GCCGAA\n",
      "\n",
      "Skipping invalid line:            GCCGAA\n",
      "\n",
      "Skipping invalid line:            GCGGAA\n",
      "\n",
      "Skipping invalid line:            GCGGAG\n",
      "\n",
      "Skipping invalid line:            GCTGAA\n",
      "\n",
      "Skipping invalid line:            GCGGAG\n",
      "\n",
      "Skipping invalid line:            GCGGAG\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Skipping invalid line:            GCGCAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000935.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000980.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000898.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001099.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000965.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001175.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001111.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001434.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            CAGGCCCTC\n",
      "\n",
      "Skipping invalid line:            AAGGCCCTG\n",
      "\n",
      "Skipping invalid line:            CGCGCCCTT\n",
      "\n",
      "Skipping invalid line:            CGCGCCCTC\n",
      "\n",
      "Skipping invalid line:            CAGGCCCTG\n",
      "\n",
      "Skipping invalid line:            AAGGCGCTT\n",
      "\n",
      "Skipping invalid line:            AAGGCGCTG\n",
      "\n",
      "Skipping invalid line:            AAGGCACTC\n",
      "\n",
      "Skipping invalid line:            AAGGCG---\n",
      "\n",
      "Skipping invalid line:            AAGGCGCTT\n",
      "\n",
      "Skipping invalid line:            AAGGCG---\n",
      "\n",
      "Skipping invalid line:            CGCGCCCTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Skipping invalid line:            AAATCATTG\n",
      "\n",
      "Skipping invalid line:            AAATCATTG\n",
      "\n",
      "Skipping invalid line:            AAATCCTTG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000938.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001072.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001147.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001183.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001160.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTCAGTCAT\n",
      "\n",
      "Skipping invalid line:            TTCCATCAC\n",
      "\n",
      "Skipping invalid line:            TTCGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTCAGCCAC\n",
      "\n",
      "Skipping invalid line:            TTTTCGCAC\n",
      "\n",
      "Skipping invalid line:            TTCGCCCAT\n",
      "\n",
      "Skipping invalid line:            TTCTCTCAC\n",
      "\n",
      "Skipping invalid line:            TTCAGCCAC\n",
      "\n",
      "Skipping invalid line:            TTCGCCCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAC\n",
      "\n",
      "Skipping invalid line:            TTCGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCCCAC\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGAGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGAGCAT\n",
      "\n",
      "Skipping invalid line:            TTTGCGCAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001211.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001127.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000825.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001437.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001224.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001388.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001425.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001220.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001266.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001397.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001165.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGC\n",
      "\n",
      "Skipping invalid line:            ACGCCGC\n",
      "\n",
      "Skipping invalid line:            ACGCCGC\n",
      "\n",
      "Skipping invalid line:            GCGTCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGC\n",
      "\n",
      "Skipping invalid line:            ACGCCGC\n",
      "\n",
      "Skipping invalid line:            GCGCCGC\n",
      "\n",
      "Skipping invalid line:            CCGCCGC\n",
      "\n",
      "Skipping invalid line:            GCGCCGC\n",
      "\n",
      "Skipping invalid line:            ACGCCGC\n",
      "\n",
      "Skipping invalid line:            GCGCCGC\n",
      "\n",
      "Skipping invalid line:            ACGCCGC\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            ACGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            ACGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Skipping invalid line:            GCGCCAT\n",
      "\n",
      "Skipping invalid line:            GCGCCGT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000890.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001136.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001381.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001257.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001174.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001173.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001364.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001121.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            ---------\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            GCGCAGGAT\n",
      "\n",
      "Skipping invalid line:            GGCAGTGAT\n",
      "\n",
      "Skipping invalid line:            GGACGGTAC\n",
      "\n",
      "Skipping invalid line:            GCCGGTGAC\n",
      "\n",
      "Skipping invalid line:            GCCGCAGAC\n",
      "\n",
      "Skipping invalid line:            GACCGCCAC\n",
      "\n",
      "Skipping invalid line:            GGCCGCGAC\n",
      "\n",
      "Skipping invalid line:            ACCCTTGAC\n",
      "\n",
      "Skipping invalid line:            CCCGAGGAC\n",
      "\n",
      "Skipping invalid line:            TCGAAAGAC\n",
      "\n",
      "Skipping invalid line:            CCCGACGAA\n",
      "\n",
      "Skipping invalid line:            GTACAGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCTGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCTGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCACGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAT\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCAGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Skipping invalid line:            CCCCCGGAC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001392.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000846.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001294.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001436.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001153.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001088.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001023.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            CA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001126.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001193.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001429.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001427.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001315.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001382.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001273.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000929.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001000.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001386.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001034.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000891.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000960.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001022.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TGCCTTAT\n",
      "\n",
      "Skipping invalid line:            TGCCTTAT\n",
      "\n",
      "Skipping invalid line:            CTTCCTTA\n",
      "\n",
      "Skipping invalid line:            CTTCCTTA\n",
      "\n",
      "Skipping invalid line:            GTTTTCTA\n",
      "\n",
      "Skipping invalid line:            CGATGCTG\n",
      "\n",
      "Skipping invalid line:            GTTTTTGC\n",
      "\n",
      "Skipping invalid line:            GCTTCCTA\n",
      "\n",
      "Skipping invalid line:            GCTTTCCA\n",
      "\n",
      "Skipping invalid line:            GTTTTCTC\n",
      "\n",
      "Skipping invalid line:            GTTTCCTA\n",
      "\n",
      "Skipping invalid line:            GCTTTCTG\n",
      "\n",
      "Skipping invalid line:            GTTTTCTG\n",
      "\n",
      "Skipping invalid line:            GTTTTTTG\n",
      "\n",
      "Skipping invalid line:            CCTTCGCA\n",
      "\n",
      "Skipping invalid line:            GTTTTCTG\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CAGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGTCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CAGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CAGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGTCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCTTTA\n",
      "\n",
      "Skipping invalid line:            TTGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTGCCTTA\n",
      "\n",
      "Skipping invalid line:            GCGCCGTA\n",
      "\n",
      "Skipping invalid line:            GCGCCTTA\n",
      "\n",
      "Skipping invalid line:            CTTCCTTA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001028.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000997.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000989.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001340.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001096.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001077.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001196.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000942.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001184.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000905.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GCGGGCGTTA \n",
      "\n",
      "Skipping invalid line:            GCGGGCGCTC \n",
      "\n",
      "Skipping invalid line:            GCGCGCGCTG \n",
      "\n",
      "Skipping invalid line:            GCGGAGTCTC \n",
      "\n",
      "Skipping invalid line:            GCGCGCGCTG \n",
      "\n",
      "Skipping invalid line:            GCGGGCGTTG \n",
      "\n",
      "Skipping invalid line:            GCGGGCGCTT \n",
      "\n",
      "Skipping invalid line:            GCGGGCGCTA \n",
      "\n",
      "Skipping invalid line:            GCGCGCCCTC \n",
      "\n",
      "Skipping invalid line:            GCGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GCGCGCGCTT \n",
      "\n",
      "Skipping invalid line:            GCGTGCGCTC \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCACTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTT \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Skipping invalid line:            GAGGGCGCTG \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001402.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001191.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000988.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001215.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001261.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001331.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001254.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001061.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000803.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001171.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000949.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001024.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001233.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001348.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001365.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001232.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001239.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001198.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000982.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000972.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000851.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000896.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001076.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001063.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001311.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001370.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001140.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001118.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001305.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001078.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001347.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000868.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001169.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001062.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            GC\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001179.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001276.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001428.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001212.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001398.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000839.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001416.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000969.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001400.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001308.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000847.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000876.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            GT\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AA\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AC\n",
      "\n",
      "Skipping invalid line:            CG\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Skipping invalid line:            AG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001314.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001356.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            --\n",
      "\n",
      "Skipping invalid line:            GA\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Skipping invalid line:            GG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001352.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001204.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001094.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GGTCGCA\n",
      "\n",
      "Skipping invalid line:            GGTCCTG\n",
      "\n",
      "Skipping invalid line:            TTGGGCA\n",
      "\n",
      "Skipping invalid line:            GGTCCAA\n",
      "\n",
      "Skipping invalid line:            GGTGCTT\n",
      "\n",
      "Skipping invalid line:            CCGAGCA\n",
      "\n",
      "Skipping invalid line:            GGTGATC\n",
      "\n",
      "Skipping invalid line:            CCAGCTA\n",
      "\n",
      "Skipping invalid line:            GATCCTA\n",
      "\n",
      "Skipping invalid line:            GGTACTT\n",
      "\n",
      "Skipping invalid line:            C---CTG\n",
      "\n",
      "Skipping invalid line:            CCCGCTA\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Skipping invalid line:            GATTATC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001101.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001310.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000999.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000865.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCCAGC\n",
      "\n",
      "Skipping invalid line:            CGTGGGC\n",
      "\n",
      "Skipping invalid line:            CGCCGGC\n",
      "\n",
      "Skipping invalid line:            GGCCGGC\n",
      "\n",
      "Skipping invalid line:            CGCAGGC\n",
      "\n",
      "Skipping invalid line:            CGCCGGA\n",
      "\n",
      "Skipping invalid line:            CGCGGGC\n",
      "\n",
      "Skipping invalid line:            CGCGGGC\n",
      "\n",
      "Skipping invalid line:            CGCGGGC\n",
      "\n",
      "Skipping invalid line:            CGCCAGC\n",
      "\n",
      "Skipping invalid line:            GGCC---\n",
      "\n",
      "Skipping invalid line:            CGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Skipping invalid line:            GGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCGGGC\n",
      "\n",
      "Skipping invalid line:            GGCAGGT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001007.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGG\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGT\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGT\n",
      "\n",
      "Skipping invalid line:            AGGTCCGG\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCCGC\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Skipping invalid line:            AGGTCGGA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001376.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001139.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CGCC\n",
      "\n",
      "Skipping invalid line:            GCCC\n",
      "\n",
      "Skipping invalid line:            CGGC\n",
      "\n",
      "Skipping invalid line:            CAGG\n",
      "\n",
      "Skipping invalid line:            ACCC\n",
      "\n",
      "Skipping invalid line:            CCCG\n",
      "\n",
      "Skipping invalid line:            CCCG\n",
      "\n",
      "Skipping invalid line:            TTCC\n",
      "\n",
      "Skipping invalid line:            CCCT\n",
      "\n",
      "Skipping invalid line:            CGAC\n",
      "\n",
      "Skipping invalid line:            GGCC\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Skipping invalid line:            CGCA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000885.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000836.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -------\n",
      "\n",
      "Skipping invalid line:            TTCCAAT\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            ATTCTCA\n",
      "\n",
      "Skipping invalid line:            CTTCTCA\n",
      "\n",
      "Skipping invalid line:            CTTCACA\n",
      "\n",
      "Skipping invalid line:            CTTTTCA\n",
      "\n",
      "Skipping invalid line:            ATTTCGA\n",
      "\n",
      "Skipping invalid line:            CTTTCGA\n",
      "\n",
      "Skipping invalid line:            CTTCCAA\n",
      "\n",
      "Skipping invalid line:            ATTTCGA\n",
      "\n",
      "Skipping invalid line:            CTTCGCA\n",
      "\n",
      "Skipping invalid line:            CTTTCAA\n",
      "\n",
      "Skipping invalid line:            TTTCACT\n",
      "\n",
      "Skipping invalid line:            CTTCTCA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCGA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Skipping invalid line:            GTTCCAA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001322.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            AAGGGC\n",
      "\n",
      "Skipping invalid line:            AACGGC\n",
      "\n",
      "Skipping invalid line:            CAGGGC\n",
      "\n",
      "Skipping invalid line:            AAGGGC\n",
      "\n",
      "Skipping invalid line:            CAGGGC\n",
      "\n",
      "Skipping invalid line:            AAGGGC\n",
      "\n",
      "Skipping invalid line:            CAGGGC\n",
      "\n",
      "Skipping invalid line:            AAGGGC\n",
      "\n",
      "Skipping invalid line:            CAGTCG\n",
      "\n",
      "Skipping invalid line:            AAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAGGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAGGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Skipping invalid line:            CAAGGC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000883.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001279.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001250.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001051.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001387.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001157.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            TAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            CAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGATC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Skipping invalid line:            GAAGTTC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001112.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001384.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CTGGAAT\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            CTGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            CTGGGAA\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            CTGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            CTGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGCAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Skipping invalid line:            ATGGGAG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001413.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000976.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000961.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001422.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000812.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001181.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001355.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000807.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000940.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001360.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001205.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000922.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            C\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            -\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            G\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            T\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Skipping invalid line:            A\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001433.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTT\n",
      "\n",
      "Skipping invalid line:            TCGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTT\n",
      "\n",
      "Skipping invalid line:            TCGGTT\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            TCGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            TCCGTG\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGCTG\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Skipping invalid line:            AAGGTC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001128.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            ATTAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAT\n",
      "\n",
      "Skipping invalid line:            ATCAAC\n",
      "\n",
      "Skipping invalid line:            GTCAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAC\n",
      "\n",
      "Skipping invalid line:            GTGAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAC\n",
      "\n",
      "Skipping invalid line:            ATCAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            ATTAAC\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Skipping invalid line:            GTGAAT\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000947.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001366.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GTGAC\n",
      "\n",
      "Skipping invalid line:            CCGCC\n",
      "\n",
      "Skipping invalid line:            AGGCG\n",
      "\n",
      "Skipping invalid line:            GC---\n",
      "\n",
      "Skipping invalid line:            CCGCT\n",
      "\n",
      "Skipping invalid line:            AGGCG\n",
      "\n",
      "Skipping invalid line:            ACCCC\n",
      "\n",
      "Skipping invalid line:            AAGGC\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            CGGCG\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            AGTCG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGT\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGT\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            ACGGG\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Skipping invalid line:            GCGGC\n",
      "\n",
      "Skipping invalid line:            GCGGC\n",
      "\n",
      "Skipping invalid line:            GCGGG\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001339.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001336.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            -----\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCAAC\n",
      "\n",
      "Skipping invalid line:            CCAAC\n",
      "\n",
      "Skipping invalid line:            CCCAA\n",
      "\n",
      "Skipping invalid line:            CGAAC\n",
      "\n",
      "Skipping invalid line:            CCGAC\n",
      "\n",
      "Skipping invalid line:            GAAAC\n",
      "\n",
      "Skipping invalid line:            GGAAT\n",
      "\n",
      "Skipping invalid line:            GAAAC\n",
      "\n",
      "Skipping invalid line:            GCAGC\n",
      "\n",
      "Skipping invalid line:            --AAC\n",
      "\n",
      "Skipping invalid line:            GGCCG\n",
      "\n",
      "Skipping invalid line:            CCAAC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCT\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Skipping invalid line:            CCCCC\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001284.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0000806.phy\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line: \n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TGACG\n",
      "\n",
      "Skipping invalid line:            TGGCG\n",
      "\n",
      "Skipping invalid line:            TCGCG\n",
      "\n",
      "Skipping invalid line:            TGACG\n",
      "\n",
      "Skipping invalid line:            TGGCG\n",
      "\n",
      "Skipping invalid line:            TGCGC\n",
      "\n",
      "Skipping invalid line:            TGGCG\n",
      "\n",
      "Skipping invalid line:            TGCGC\n",
      "\n",
      "Skipping invalid line:            TGGCC\n",
      "\n",
      "Skipping invalid line:            TCGCC\n",
      "\n",
      "Skipping invalid line:            TGGCC\n",
      "\n",
      "Skipping invalid line:            CGGCG\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Skipping invalid line:            TCACA\n",
      "\n",
      "Modified file saved as /mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise/OG0001255.phy\n"
     ]
    }
   ],
   "source": [
    "### name revise \n",
    "def modify_phylip_names(input_file, output_file):\n",
    "    with open(input_file, 'r') as infile:\n",
    "        # 读取第一行，它包含序列数和序列长度\n",
    "        header = infile.readline().strip()\n",
    "\n",
    "        # 读取剩余的序列行\n",
    "        sequences = infile.readlines()\n",
    "\n",
    "    # 打开输出文件，准备写入修改后的内容\n",
    "    with open(output_file, 'w') as outfile:\n",
    "        # 首先写入头部信息\n",
    "        outfile.write(header + '\\n')\n",
    "\n",
    "        # 遍历每条序列\n",
    "        for seq in sequences:\n",
    "            # 分割序列名称和序列部分\n",
    "            parts = seq.strip().split(maxsplit=1)\n",
    "            if len(parts) != 2:\n",
    "                print(f\"Skipping invalid line: {seq}\")\n",
    "                continue\n",
    "\n",
    "            name, sequence = parts\n",
    "            \n",
    "            # 获取名称的后8个字符，保留序列部分\n",
    "            new_name = name[-8:]\n",
    "            \n",
    "            # 确保名称和序列之间有至少2个空格，这对 Phylip 格式是强制要求的\n",
    "            outfile.write(f\"{new_name:<10}  {sequence}\\n\")\n",
    "\n",
    "    print(f\"Modified file saved as {output_file}\")\n",
    "\n",
    "def get_revise_phy(raw_folder,new_folder):\n",
    "    try:\n",
    "        os.makedirs(new_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(new_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(raw_folder):\n",
    "        file_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(raw_folder, filename)\n",
    "        modify_phylip_names(file_path, f\"{new_folder}/{file_name}.phy\")\n",
    "get_revise_phy(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy\",\"/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_revise\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "3de0ea67-694a-42f2-bb54-ff48165cb8e3",
   "metadata": {},
   "outputs": [],
   "source": [
    "###run PAML to get dN/dS\n",
    "def prepare_ctl_run(codon_align_phy,ctl_file,target_file):\n",
    "    content = [f\"seqfile = {codon_align_phy}  * sequence data file name\", f\"outfile = {target_file}       * main result file\", \"verbose = 0  * 1: detailed output (list sequences), 0: concise output \", \"icode = 0  * 0:universal code; 1:mammalian mt; 2-10:see below \",\"weighting = 0  * weighting pathways between codons (0/1)? \",\"commonf3x4 = 0  * use one set of codon freqs for all pairs (0/1)?\"]\n",
    "    with open(ctl_file, 'w') as file:\n",
    "        for line in content:\n",
    "            file.write(line + '\\n\\n')\n",
    "    os.system(f\"/mnt/maple/ymchen/paml/bin/yn00 {ctl_file} > /dev/null 2>&1\")\n",
    "    \n",
    "def PAML_workflow(phy_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    os.system(f\"cd {target_folder}\")\n",
    "    for filename in os.listdir(phy_folder):\n",
    "        file_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(phy_folder, filename)\n",
    "        prepare_ctl_run(file_path,f\"{target_folder}/yn00.ctl\",f\"{target_folder}/{file_name}.out\")\n",
    "PAML_workflow(\"/mnt/maple/ymchen/gene_loss/preflow/codon_align_phy_new\",\"/mnt/maple/ymchen/gene_loss/preflow/yn00_result\")    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "04b38cf8-3310-4f90-acfc-634d7a56b730",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000899\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001383\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001149\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000916\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001140\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001430\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001391\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001121\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001186\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001227\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001339\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001000\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000874\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001306\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000895\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001342\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001061\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001208\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001402\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001139\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001238\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001290\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001337\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001250\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001007\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001170\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000909\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001133\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001322\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000829\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000938\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000842\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001100\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001219\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000980\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001152\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001377\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001063\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001076\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000969\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001042\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000990\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001270\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001321\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001316\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001053\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000905\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000966\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001412\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001092\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001289\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000877\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001307\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001107\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001110\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001058\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000891\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000972\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001287\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000886\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001217\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001201\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001356\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001286\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001424\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001033\n",
      "未找到任何 dN/dS 结果或基因名称对。\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000870\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001409\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000866\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001410\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001429\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001067\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000995\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001344\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000963\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000849\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001106\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000817\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001134\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001089\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001190\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000898\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001101\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001401\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001128\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000862\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001318\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001109\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001212\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001355\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001179\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000930\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000925\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001088\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000856\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001019\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000846\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001173\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001367\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001116\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001029\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001341\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001413\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001125\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001094\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001122\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001425\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000964\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001335\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000863\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001035\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000935\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000881\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001320\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000965\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001338\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001210\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001054\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001187\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001241\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000859\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001017\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001319\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001385\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001397\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001127\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001204\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001359\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001396\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001051\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000848\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001398\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001165\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001327\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000949\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001235\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001209\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001292\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000873\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001328\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001278\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001411\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001288\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001087\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000967\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001272\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001384\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000812\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001378\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000867\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001200\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001203\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000897\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001117\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001145\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001192\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001313\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001223\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001309\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000947\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001263\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000840\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001070\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001078\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001010\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001059\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000839\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000835\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001374\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001254\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000939\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001285\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001386\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001220\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001426\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000954\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001276\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001115\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001077\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001079\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001358\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001031\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000975\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001018\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001269\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001257\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001421\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001232\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000941\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000924\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001434\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001253\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001277\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001034\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001193\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000892\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001291\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000999\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001437\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001011\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000908\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001394\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001147\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000951\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001297\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001294\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001162\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001091\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001174\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001214\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001299\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001169\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000831\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001259\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000878\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001244\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001164\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001387\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001243\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001175\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001351\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000953\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001281\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001197\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001380\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001280\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001037\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000869\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000845\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001153\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001020\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000826\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001112\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000823\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001264\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000943\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001365\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001081\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000984\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000940\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001376\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001118\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001004\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000914\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001027\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001242\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000976\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001136\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001119\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001392\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000901\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000847\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001151\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001298\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001068\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000805\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001345\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001066\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001382\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001105\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001370\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001262\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000993\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001069\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000981\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000838\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001255\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001438\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000957\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001168\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001271\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001366\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000868\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001331\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001211\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000987\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001161\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001304\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000880\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000857\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000879\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001245\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001311\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001222\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001108\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000921\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000950\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000825\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000906\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001369\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001184\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001159\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001302\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001016\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000994\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001352\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000919\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001361\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001144\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000913\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000804\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001347\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001418\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001130\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001206\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001181\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001008\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001009\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001177\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000860\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000946\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001416\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001314\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001221\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000888\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001340\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000917\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000944\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001408\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001375\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001379\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001273\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001196\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001301\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000883\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001099\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001049\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000896\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000822\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001414\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001172\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001308\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000836\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001336\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001137\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000882\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001310\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001282\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000974\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001323\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001215\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001417\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000926\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001317\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000982\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001072\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001138\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001026\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001126\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001404\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000937\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001423\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000853\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001274\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001431\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001205\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001260\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001120\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000803\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000923\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001283\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000912\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001373\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001330\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001239\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001065\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001360\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001249\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000968\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000871\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001432\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001103\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001315\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001086\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001218\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000989\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001216\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001354\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001001\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000832\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000806\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000875\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000931\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001156\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001132\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001182\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001265\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001229\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001415\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000893\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001002\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001312\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001056\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001225\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000920\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000902\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000851\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001433\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000834\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001399\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001390\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001261\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001178\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001207\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001326\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000996\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001090\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000973\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001064\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001025\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001158\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001266\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001395\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000890\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001028\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000855\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000807\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000816\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001279\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000961\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000858\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001062\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001439\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001436\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001228\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000876\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000998\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000932\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001102\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001155\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001403\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001198\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000979\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001348\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001388\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000997\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000988\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001036\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001043\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000833\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001113\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001284\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001233\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001183\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000915\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001258\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001023\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000929\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001074\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001350\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001333\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001085\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001188\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001329\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001045\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001332\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001442\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000960\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001252\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001096\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001124\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001364\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000977\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001050\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000872\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001226\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000927\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001368\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001114\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001071\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001353\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001199\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001428\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001129\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001305\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000837\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001171\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001324\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001095\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001080\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001191\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001157\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001131\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001240\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001440\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001422\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001248\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001381\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001343\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001057\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001135\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001024\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001346\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001389\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001363\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001055\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000956\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001224\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001111\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001334\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001093\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001142\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000942\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001419\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000955\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001189\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001022\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001427\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000962\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001123\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001295\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000885\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001160\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001097\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001400\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000922\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001237\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0000865\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001420\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001268\n",
      "dN/dS 结果已保存到 /mnt/maple/ymchen/gene_loss/preflow/dNdS_result/OG0001293\n"
     ]
    }
   ],
   "source": [
    "###calculate result from PAML output\n",
    "import re\n",
    "import pandas as pd\n",
    "\n",
    "def extract_dnds(yn_out, output_file):\n",
    "    with open(yn_out, 'r') as f:\n",
    "        lines = f.readlines()\n",
    "    dnds_data = []\n",
    "    name_pairs = []\n",
    "    dnds_pattern = re.compile(r'^\\s+\\d+\\s+\\d+\\s+(\\S+)\\s+(\\S+)\\s+\\S+\\s+\\S+\\s+\\S+\\s+(\\S+)\\s+\\+\\-\\s+\\S+\\s+(\\S+)\\s+\\+\\-')\n",
    "    name_pattern = re.compile(r'(\\S+)\\s+vs\\.\\s+(\\S+)')\n",
    "    for line in lines:\n",
    "        dnds_match = dnds_pattern.match(line)\n",
    "        if dnds_match:\n",
    "            gene1, gene2, dN, dS = dnds_match.groups()\n",
    "            dnds_data.append([gene1, gene2, dN, dS])\n",
    "        name_match = name_pattern.search(line)\n",
    "        if name_match:\n",
    "            gene1_name, gene2_name = name_match.groups()\n",
    "            name_pairs.append([gene1_name, gene2_name])\n",
    "    if dnds_data and name_pairs:\n",
    "        result_data = []\n",
    "        for i in range(len(dnds_data)):\n",
    "            result_data.append(name_pairs[i] + dnds_data[i][2:])  # 合并名称对和dN/dS值\n",
    "        df = pd.DataFrame(result_data, columns=[\"gene1\", \"gene2\", \"dN\", \"dS\"])\n",
    "        df.to_csv(output_file, sep='\\t', index=False)\n",
    "        print(f\"dN/dS 结果已保存到 {output_file}\")\n",
    "    else:\n",
    "        print(\"未找到任何 dN/dS 结果或基因名称对。\")\n",
    "def dNdS_workflow(yn00_folder,target_folder):\n",
    "    try:\n",
    "        os.makedirs(target_folder, exist_ok=True)\n",
    "    except OSError as err:\n",
    "        if err.errno == os.errno.EEXIST and os.path.isdir(target_folder):\n",
    "            pass\n",
    "        else:\n",
    "            raise\n",
    "    for filename in os.listdir(yn00_folder):\n",
    "        file_name=filename.rsplit(\".\",1)[0]\n",
    "        file_path = os.path.join(yn00_folder, filename)\n",
    "        extract_dnds(file_path, f\"{target_folder}/{file_name}\")\n",
    "dNdS_workflow(\"/mnt/maple/ymchen/gene_loss/preflow/yn00_result\", \"/mnt/maple/ymchen/gene_loss/preflow/dNdS_result\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dd69f8c1-d7e8-4643-bfc3-685d246e2d2b",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
